>Table S2. A comprehensive list of interactive biological web applications developed using R/Shiny.  

| Year | Name                                         | URL                                                                                        | Source code                                                                                                       | DOI                               | Journal                                                           | Title                                                                                                                                                                                                    |
|------|----------------------------------------------|--------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------|-----------------------------------|-------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| 2013 | diveRsity                                    | http://glimmer.rstudio.com/kkeenan/diveRsity-online/                                       | https://cran.r-project.org/web/packages/diveRsity/index.html                                                      | 10.1111/2041-210X.12067           | Methods in Ecology and Evolution                                  | diveRsity: An R package for the estimation and exploration of population genetics parameters and their associated errors                                                                                 |
| 2014 | BoxPlotR                                     | http://shiny.chemgrid.org/boxplotr/                                                        | https://github.com/VizWizard/BoxPlotR.shiny                                                                       | 10.1038/nmeth.2811                | Nature Methods                                                    | BoxPlotR: a web tool for generation of box plots                                                                                                                                                         |
| 2014 | MethylAid                                    | NA                                                                                         | www.bioconductor.org/packages/3.0/bioc/html/MethylAid.html                                                        | 10.1093/bioinformatics/btu566     | Bioinformatics                                                    | MethylAid: visual and interactive quality control of large Illumina 450k datasets                                                                                                                        |
| 2014 | RNAseqPS                                     | http://cqs.mc.vanderbilt.edu/shiny/RNAseqPS/                                               | NA                                                                                                                | 10.4137/CIN.S17688                | Cancer Informatics                                                | RNAseqPS: A Web Tool for Estimating Sample Size and Power for RNAseq Experiment                                                                                                                          |
| 2014 | Shiny-phyloseq                               | http://joey711.github.io/shiny-phyloseq/                                                   | https://github.com/joey711/shiny-phyloseq                                                                         | 10.1093/bioinformatics/btu616     | Bioinformatics                                                    | Shiny-phyloseq: Web application for interactive microbiome analysis with provenance tracking                                                                                                             |
| 2014 | strplot                                      | http://btismysore.in/strplot                                                               | NA                                                                                                                | 10.1186/2193-1801-3-431           | SpringerPlus                                                      | STRUCTURE PLOT: a program for drawing elegant STRUCTURE bar plots in user friendly interface                                                                                                             |
| 2014 | SumAOV1                                      | http://shiny.stat.tamu.edu:3838/hassaad/SumAOV1/                                           | NA                                                                                                                | 10.1186/2193-1801-3-474           | SpringerPlus                                                      | Rapid publication-ready MS-Word tables for one-way ANOVA                                                                                                                                                 |
| 2015 | chipPCR                                      | NA                                                                                         | https://github.com/michbur/chipPCR                                                                                | 10.1093/bioinformatics/btv205     | Bioinformatics                                                    | chipPCR: an R package to pre-process raw data of amplification curves                                                                                                                                    |
| 2015 | coMET                                        | http://epigen.kcl.ac.uk/comet/                                                             | http://epigen.kcl.ac.uk/comet/                                                                                    | 10.1186/s12859-015-0568-2         | BMC Bioinformatics                                                | coMET: visualisation of regional epigenome-wide association scan results and DNA co-methylation patterns                                                                                                 |
| 2015 | Crambled                                     | https://github.com/dralynch/crambled                                                       | https://dralynch.shinyapps.io/crambled_app                                                                        | 10.12688/f1000research.7453.1     | F1000Research                                                     | Crambled: A Shiny application to enable intuitive resolution of conflicting cellularity estimates                                                                                                        |
| 2015 | Goplot                                       | https://wwalter.shinyapps.io/Venn/                                                         | https://wencke.github.io/                                                                                         | 10.1093/bioinformatics/btv300     | Bioinformatics                                                    | GOplot: an R package for visually combining expression data with functional analysis                                                                                                                     |
| 2015 | MagiCMicroRna                                | NA                                                                                         | https://bitbucket.org/mutgx/magicmicrorna.git                                                                     | 10.1186/s13029-015-0035-5         | Source Code for Biology and Medicine                              | MagiCMicroRna: a web implementation of AgiMicroRna using shiny                                                                                                                                           |
| 2015 | MAVIS                                        | http://kylehamilton.net/shiny/MAVIS/                                                       | https://github.com/kylehamilton/MAVIS                                                                             | 10.13140/RG.2.1.3316.7205         | Technical Report                                                  | MAVIS: Meta Analysis via Shiny                                                                                                                                                                           |
| 2015 | MetaMapR                                     | http://dgrapov.github.io/MetaMapR/                                                         | http://dgrapov.github.io/MetaMapR/                                                                                | 10.1093/bioinformatics/btv194     | Bioinformatics                                                    | MetaMapR: pathway independent metabolomic network analysis incorporating unknowns                                                                                                                        |
| 2015 | Peax                                         | http://compbio.ucdenver.edu/PEAX                                                           | NA                                                                                                                | 10.1142/9789814644730_0040        | Pacific Symposium on Biocomputing                                 | Peax: interactive visual analysis and exploration of complex clinical phenotype and gene expression association                                                                                          |
| 2015 | pmsignature                                  | https://friend1ws.shinyapps.io/pmsignature_shiny/                                          | https://github.com/friend1ws/pmsignature                                                                          | 10.1371/journal.pgen.1005657      | PLoS genetics                                                     | A Simple Model-Based Approach to Inferring and Visualizing Cancer Mutation Signatures                                                                                                                    |
| 2015 | Rchemcpp                                     | http://shiny.bioinf.jku.at/Analoging/                                                      | NA                                                                                                                | 10.1093/bioinformatics/btv373     | Bioinformatics                                                    | Rchemcpp: a web service for structural analoging in ChEMBL, Drugbank and the Connectivity Map                                                                                                            |
| 2015 | RxODE                                        | http://qsp.engr.uga.edu:3838/RxODE/RegimenSimulator                                        | https://github.com/hallowkm/RxODE                                                                                 | 10.1002/psp4.12052                | CPT: Pharmacometrics & Systems Pharmacology                       | A Tutorial on RxODE: Simulating Differential Equation Pharmacometric Models in R                                                                                                                         |
| 2015 | Seed                                         | NA                                                                                         | https://github.com/danlbek/Seed                                                                                   | 10.1093/bioinformatics/btu693     | Bioinformatics                                                    | Seed: a user-friendly tool for exploring and visualizing microbial community data                                                                                                                        |
| 2015 | SynRio                                       | http://www.nfmc.res.in/synrio/                                                             | https://github.com/NFMC/SynRio                                                                                    | 10.6026/97320630011422            | Bioinformation                                                    | SynRio: R and Shiny based application platform for cyanobacterial genome analysis                                                                                                                        |
| 2015 | Thinking Outside the Box                     | https://sites.google.com/site/psychvisualizations/                                         | NA                                                                                                                | 10.3389/fpsyg.2015.01782          | Frontiers in psychology                                           | Thinking Outside the Box: Developing Dynamic Data Visualizations for Psychology with Shiny                                                                                                               |
| 2016 | AdvISER-PYRO-SMQ                             | https://ucl-irec-ctma.shinyapps.io/Pyrosequencing-NRAS-61                                  | NA                                                                                                                | 10.1186/s13015-016-0086-4         | Algorithms for molecular biology                                  | Using a constraint-based regression method for relative quantification of somatic mutations in pyrosequencing signals: a case for NRAS analysis                                                          |
| 2016 | AlignStat                                    | http://alignstat.science.latrobe.edu.au/                                                   | https://github.com/iracooke/AlignStatShiny                                                                        | 10.1186/s12859-016-1300-6         | BMC Bioinformatics                                                | AlignStat: a web-tool and R package for statistical comparison of alternative multiple sequence alignments                                                                                               |
| 2016 | APOSTL                                       | http://apostl.moffitt.org/                                                                 | https://github.com/bornea/APOSTL                                                                                  | 10.1021/acs.jproteome.6b00660     | Journal of Proteome Research                                      | APOSTL: An Interactive Galaxy Pipeline for Reproducible Analysis of Affinity Proteomics Data                                                                                                             |
| 2016 | Bio3D-web                                    | http://thegrantlab.org/bio3d-web                                                           | NA                                                                                                                | 10.1093/bioinformatics/btw482     | Bioinformatics                                                    | Online interactive analysis of protein structure ensembles with Bio3D-web                                                                                                                                |
| 2016 | CLME                                         | NA                                                                                         | https://github.com/jelsema/CLME                                                                                   | 10.18637/jss.v075.i01             | Journal of statistical software                                   | CLME: An R Package for Linear Mixed Effects Models under Inequality Constraints                                                                                                                          |
| 2016 | Cytofkit                                     | NA                                                                                         | https://bioconductor.org/packages/cytofkit/                                                                       | 10.1371/journal.pcbi.1005112      | PLoS Computational Biology                                        | Cytofkit: A Bioconductor Package for an Integrated Mass Cytometry Data Analysis Pipeline                                                                                                                 |
| 2016 | ddpcr                                        | https://daattali.com/shiny/ddpcr/                                                          | https://github.com/daattali/ddpcr                                                                                 | 10.12688/f1000research.9022.1     | F1000Research                                                     | ddpcr: an R package and web application for analysis of droplet digital PCR data.                                                                                                                        |
| 2016 | ddPCRmulti                                   | http://bit.ly/ddPCRmulti                                                                   | NA                                                                                                                | 10.1038/srep35451                 | Scientific Reports                                                | Multiplex quantification of four DNA targets in one reaction with Bio-Rad droplet digital PCR system for GMO detection                                                                                   |
| 2016 | DotMapper                                    | https://github.com/cathsmith57/DotMapper                                                   | NA                                                                                                                | 10.1186/s12879-016-1475-5         | BMC infectious diseases                                           | DotMapper: an open source tool for creating interactive disease point maps                                                                                                                               |
| 2016 | easyROC                                      | http://www.biosoft.hacettepe.edu.tr/easyROC/                                               | http://www.github.com/dncR/easyROC                                                                                | 10.32614/RJ-2016-042              | The R Journal                                                     | easyROC: An Interactive Web-tool for ROC Curve Analysis Using R Language Environment                                                                                                                     |
| 2016 | EHDViz                                       | http://ehdviz.dudleylab.org                                                                | NA                                                                                                                | 10.1136/bmjopen-2015-010579       | BMJ open                                                          | EHDViz: clinical dashboard development using open-source technologies                                                                                                                                    |
| 2016 | GAM                                          | https://artyomovlab.wustl.edu/shiny/gam/                                                   | https://github.com/ctlab/shinygam                                                                                 | 10.1093/nar/gkw266                | Nucleic Acids Research                                            | GAM: a web-service for integrated transcriptional and metabolic network analysis                                                                                                                         |
| 2016 | hemaClass                                    | NA                                                                                         | https://github.com/oncoclass/hemaclass                                                                            | 10.1371/journal.pone.0163711      | Plos One                                                          | hemaClass.org: Online One-By-One Microarray Normalization and Classification of Hematological Cancers for Precision Medicine                                                                             |
| 2016 | IncucyteDRC                                  | NA                                                                                         | https://github.com/chapmandu2/IncucyteDRC/                                                                        | 10.12688/f1000research.8694.1     | F1000Research                                                     | IncucyteDRC: An R package for the dose response analysis of live cell imaging data                                                                                                                       |
| 2016 | MAX UnMix                                    | http://www.irm.umn.edu/maxunmix                                                            | https://github.com/dan-maxbauer/max-unmix                                                                         | 10.1016/j.cageo.2016.07.009       | Computers & Geosciences                                           | MAX UnMix: A web application for unmixing magnetic coercivity distributions                                                                                                                              |
| 2016 | MEMHDX                                       | http://memhdx.c3bi.pasteur.fr                                                              | NA                                                                                                                | 10.1093/bioinformatics/btw420     | Bioinformatics                                                    | MEMHDX: an interactive tool to expedite the statistical validation and visualization of large HDX-MS datasets                                                                                            |
| 2016 | Methylation plotter                          | http://gattaca.imppc.org:3838/methylation_plotter/                                         | NA                                                                                                                | 10.1186/1751-0473-9-11            | Source Code for Biology and Medicine                              | Methylation plotter: a web tool for dynamic visualization of DNA methylation data                                                                                                                        |
| 2016 | MicroScope                                   | http://microscopebioinformatics.org/                                                       | https://github.com/Bohdan-Khomtchouk/Microscope                                                                   | 10.1186/s12859-016-1260-x         | BMC Bioinformatics                                                | MicroScope: ChIP-seq and RNA-seq software analysis suite for gene expression heatmaps                                                                                                                    |
| 2016 | Milieu Interieur                             | https://www.synapse.org/MilieuInterieur                                                    | NA                                                                                                                | 10.1016/j.celrep.2016.08.011      | Cell reports                                                      | Standardized Whole-Blood Transcriptional Profiling Enables the Deconvolution of Complex Induced Immune Responses                                                                                         |
| 2016 | msVolcano                                    | http://projects.biotec.tu-dresden.de/msVolcano/                                            | NA                                                                                                                | 10.1002/pmic.201600167            | Proteomics                                                        | msVolcano: A flexible web application for visualizing quantitative proteomics data                                                                                                                       |
| 2016 | pophelper                                    | http://pophelper.com/                                                                      | https://github.com/royfrancis/pophelper                                                                           | 10.1111/1755-0998.12509           | Molecular Ecology Resources                                       | pophelper: an R package and web app to analyse and visualize population structure                                                                                                                        |
| 2016 | Proof-of-Principle of Rtlc                   | http://shinyapps.ernaehrung.unigiessen.de/rtlc                                             | https://github.com/DimitriF/rTLC-apps                                                                             | 10.1021/acs.analchem.6b04017      | Analytical Chemistry                                              | Proof-of-Principle of rTLC, an Open-Source Software Developed for Image Evaluation and Multivariate Analysis of Planar Chromatograms                                                                     |
| 2016 | rcellminer                                   | NA                                                                                         | http://www.bioconductor.org/packages/rcellminer                                                                   | 10.1093/bioinformatics/btv701     | Bioinformatics                                                    | rcellminer: exploring molecular profiles and drug response of the NCI-60 cell lines in R                                                                                                                 |
| 2016 | RevEcoR                                      | http://yiluheihei.shinyapps.io/shiny-RevEcoR                                               | https://github.com/yiluheihei/shiny-RevEcoR                                                                       | 10.1186/s12859-016-1088-4         | BMC Bioinformatics                                                | RevEcoR: an R package for the reverse ecology analysis of microbiomes                                                                                                                                    |
| 2016 | ShinyMDE                                     | https://hussain.shinyapps.io/App-1                                                         | https://github.com/AgazW/shinyMDE                                                                                 | 10.1109/BSB.2016.7552152          | IEEE                                                              | ShinyMDE: Shiny tool for microarray meta-analysis for differentially expressed gene detection                                                                                                            |
| 2016 | VennDiagramWeb                               | http://venndiagram.res.oicr.on.ca/                                                         | NA                                                                                                                | 10.1186/s12859-016-1281-5         | BMC Bioinformatics                                                | VennDiagramWeb: a web application for the generation of highly customizable Venn and Euler diagrams                                                                                                      |
| 2017 | ABI Base Recall                              | https://pasteurmaroc.shinyapps.io/abi_base_recall_app/                                     | https://github.com/zikozak92/ABI-Base-Recall-App                                                                  | 10.1109/TNB.2017.2755004          | IEEE transactions on nanobioscience                               | ABI Base Recall: Automatic Correction and Ends Trimming of DNA Sequences                                                                                                                                 |
| 2017 | adat                                         | https://foocheung.shinyapps.io/adat/                                                       | https://github.com/foocheung/adat                                                                                 | 10.5334/jors.166                  | Journal of Open Research Software                                 | Web Tool for Navigating and Plotting SomaLogic ADAT Files                                                                                                                                                |
| 2017 | AFCMHeatMap                                  | https://mohammadtarek.shinyapps.io/AFCMHeatMap/                                            | https://github.com/MTarekM/AFCMHeatMapGeneratingTool                                                              | 10.7287/peerj.preprints.2961v2    | PeerJ                                                             | A Shiny web Tool for HeatMap Generation of genetic expression datasets using R packages                                                                                                                  |
| 2017 | ALTRE                                        | NA                                                                                         | https://github.com/Mathelab/ALTRE                                                                                 | 10.1093/bioinformatics/btw688     | Bioinformatics                                                    | ALTRE: workflow for defining ALTered Regulatory Elements using chromatin accessibility data                                                                                                              |
| 2017 | AmyloGram                                    | http://smorfland.uni.wroc.pl/shiny/AmyloGram/                                              | https://github.com/michbur/AmyloGramAnalysis                                                                      | 10.1038/s41598-017-13210-9        | Scientific Reports                                                | Amyloidogenic motifs revealed by n-gram analysis                                                                                                                                                         |
| 2017 | animalnexus                                  | http://animalnexus.ca/                                                                     | https://github.com/animalnexus/animalnexus.github.io                                                              | 10.1002/ece3.3240                 | Ecology and Evolution                                             | feedr and animalnexus.ca: A paired R package and user‐friendly Web application for transforming and visualizing animal movement data from static stations                                                |
| 2017 | binQTL                                       | https://venyao.xyz/binQTL.shiny/                                                           | https://github.com/venyao/binQTL.shiny                                                                            | 10.1016/j.jgg.2019.06.005         | Journal of Genetics and Genomics                                  | binQTL: an R package for QTL mapping with binmap data                                                                                                                                                    |
| 2017 | brain-coX                                    | http://shiny.bioinf.wehi.edu.au/freytag.s/                                                 | http://human.brain-map.org/static/download                                                                        | 10.1186/s13073-017-0444-y         | Genome Medicine                                                   | brain-coX: investigating and visualising gene co-expression in seven human brain transcriptomic datasets.                                                                                                |
| 2017 | Caffsim                                      | https://asan.shinyapps.io/caff                                                             | https://github.com/asancpt/shiny-caff                                                                             | 10.12793/tcp.2017.25.3.141        | Translational and clinical pharmacology                           | Caffsim: simulation of plasma caffeine concentrations implemented as an R package and Web-applications                                                                                                   |
| 2017 | camcAPP                                      | http://bioinformatics.cruk.cam.ac.uk/apps/camcAPP/                                         | NA                                                                                                                | 10.1016/j.ebiom.2017.02.022       | EBioMedicine                                                      | Mining Human Prostate Cancer Datasets: The “camcAPP” Shiny App                                                                                                                                           |
| 2017 | CASAS                                        | http://shinygispa.winship.emory.edu/CASAS/                                                 | https://github.com/manalirupji/CASAS/                                                                             | 10.12688/f1000research.11830.2    | F1000Research                                                     | CASAS: Cancer Survival Analysis Suite, a web based application                                                                                                                                           |
| 2017 | Cerrado                                      | http://cyto.shinyapps.io/cerrado                                                           | NA                                                                                                                | 10.3897/CompCytogen               | Comparative cytogenetics                                          | The Cerrado (Brazil) plant cytogenetics database                                                                                                                                                         |
| 2017 | Condir                                       | https://utrecht-university.shinyapps.io/Condir/                                            | https://github.com/AngelosPsy/condir                                                                              | 10.1080/20008198.2017.1314782     | European journal of psychotraumatology                            | Bayesian hypothesis testing for human threat conditioning research: an introduction and the condir R package                                                                                             |
| 2017 | CVE                                          | NA                                                                                         | https://bioconductor.org/packages/CVE/                                                                            | 10.1186/s12920-017-0261-6         | BMC medical genomics                                              | CVE: an R package for interactive variant prioritisation in precision oncology                                                                                                                           |
| 2017 | DEApp                                        | https://yanli.shinyapps.io/DEApp                                                           | https://github.com/yan-cri/DEApp                                                                                  | 10.1186/s13029-017-0063-4         | Source Code for Biology and Medicine                              | DEApp: an interactive web interface for differential expression analysis of next generation sequence data                                                                                                |
| 2017 | EMSaov                                       | NA                                                                                         | https://cran.r-project.org/web/packages/EMSaov/index.html                                                         | 10.32614/rj-2017-011              | The R Journal                                                     | EMSaov: An R Package for the Analysis of Variance with the Expected Mean Squares and its Shiny Application                                                                                               |
| 2017 | ExpressionDB                                 | https://www.shinyapps.io/                                                                  | https://github.com/5c077/ExpressionDB/issues                                                                      | 10.1371/journal.pone.0187457      | Plos One                                                          | ExpressionDB: An open source platform for distributing genome-scale datasets                                                                                                                             |
| 2017 | GAC                                          | http://shinygispa.winship.emory.edu/GAC/                                                   | https://github.com/manalirupji/GAC                                                                                | 10.12688/f1000research.11840.2    | F1000Research                                                     | GAC: Gene Associations with Clinical, a web based application                                                                                                                                            |
| 2017 | GalaxyDocker                                 | NA                                                                                         | https://github.com/CARPEM/GalaxyDocker                                                                            | 10.1093/gigascience/gix099        | GigaScience                                                       | An architecture for genomics analysis in a clinical setting using Galaxy and Docker                                                                                                                      |
| 2017 | GRcalculator                                 | http://www.grcalculator.org/                                                               | https://github.com/datarail/gr_metrics                                                                            | 10.1186/s12885-017-3689-3         | BMC Cancer                                                        | GRcalculator: an online tool for calculating and mining dose–response data                                                                                                                               |
| 2017 | gwasMP                                       | http://cnsgenomics.com/shiny/gwasMP                                                        | https://github.com/jianyangqt/gwasMP/                                                                             | 10.1186/s13059-017-1216-0         | Genome Biology                                                    | Quantifying the mapping precision of genome-wide association studies using whole-genome sequencing data                                                                                                  |
| 2017 | heatmaply                                    | NA                                                                                         | https://github.com/talgalili//heatmaplyExamples                                                                   | 10.1093/bioinformatics/btx657     | Bioinformatics                                                    | heatmaply: an R package for creating interactive cluster heatmaps for online publishing                                                                                                                  |
| 2017 | HMDPDatabase                                 | https://systems.genetics.ucla.edu/                                                         | https://github.com/ChristophRau/HMDPDatabase                                                                      | 10.1007/978-1-4939-6427-7_7       | Methods in Molecular Biology                                      | A Suite of Tools for Biologists That Improve Accessibility and Visualization of Large Systems Genetics Datasets: Applications to the Hybrid Mouse Diversity Panel                                        |
| 2017 | id_artemisinin_resistance                    | https://malaria.shinyapps.io/id_artemisinin_resistance/                                    | https://github.com/SaiTheinThanTun/mixmodelpaperfinal                                                             | 10.1371/journal.pone.0177840      | Plos One                                                          | Identifying artemisinin resistance from parasite clearance half-life data with a simple Shiny web application                                                                                            |
| 2017 | Intervene                                    | https://asntech.shinyapps.io/intervene/                                                    | https://bitbucket.org/CBGR/intervene/src/master/                                                                  | 10.1186/s12859-017-1708-7         | BMC Bioinformatics                                                | Intervene: a tool for intersection and visualization of multiple gene or genomic region sets                                                                                                             |
| 2017 | IVAG                                         | https://hub.docker.com/r/leetaerim/ivag/                                                   | https://github.com/jmoa/IVAG                                                                                      | 10.5808/GI.2017.15.4.178          | Genomics & Informatics                                            | IVAG: An Integrative Visualization Application for Various Types of Genomic Data Based on R-Shiny and the Docker Platform                                                                                |
| 2017 | MALDIViz                                     | https://jkkishore85.shinyapps.io/maldiviz/                                                 | NA                                                                                                                | 10.1177/2472555217727517          | SLAS discovery                                                    | MALDIViz: A Comprehensive Informatics Tool for MALDI-MS Data Visualization and Analysis                                                                                                                  |
| 2017 | MareyMap Online                              | http://lbbe-shiny.univ-lyon1.fr/MareyMapOnline/                                            | https://github.com/aursiber/MareyMap                                                                              | 10.1093/gbe/evx178                | Genome Biology and Evolution                                      | MareyMap Online: A User-Friendly Web Application and Database Service for Estimating Recombination Rates Using Physical and Genetic Maps                                                                 |
| 2017 | MetaGenyo                                    | http://bioinfo.genyo.es/metagenyo/                                                         | NA                                                                                                                | 10.1186/s12859-017-1990-4         | BMC Bioinformatics                                                | MetaGenyo: a web tool for meta-analysis of genetic association studies                                                                                                                                   |
| 2017 | Method Comparison                            | https://bahar.shinyapps.io/method_compare/                                                 | NA                                                                                                                | 10.1016/j.clinbiochem.2017.08.008 | Clinical biochemistry                                             | An interactive website for analytical method comparison and bias estimation                                                                                                                              |
| 2017 | miRcomp-Shiny                                | https://laurenkemperman.shinyapps.io/mircomp/                                              | https://doi.org/10.5281/zenodo.1049074                                                                            | 10.12688/f1000research.13098.1    | F1000Research                                                     | miRcomp-Shiny: Interactive assessment of qPCR-based microRNA quantification and quality control algorithms                                                                                               |
| 2017 | miRNAmeConverter                             | http://www.systemsmedicineireland.ie/tools/mirna-name-converter/                           | http://bioconductor.org/packages/miRBaseVersions.db                                                               | 10.1093/bioinformatics/btw660     | Bioinformatics                                                    | miRNAmeConverter: an R/bioconductor package for translating mature miRNA names to different miRBase versions                                                                                             |
| 2017 | NMRProcFlow                                  | http://nmrprocflow.org                                                                     | NA                                                                                                                | 10.1007/s11306-017-1178-y         | Metabolomics                                                      | NMRProcFlow: a graphical and interactive tool dedicated to 1D spectra processing for NMR-based metabolomics                                                                                              |
| 2017 | nonparbnch                                   | https://uvatrapps.shinyapps.io/nonparbnch/                                                 | http://faculty.virginia.edu/model-based_dose-finding/                                                             | 10.1177/1740774517715456          | Clinical Trials                                                   |  A web application for evaluating Phase I methods using a non-parametric optimal benchmark                                                                                                               |
| 2017 | P-CAPS                                       | http://panflam.korea.ac.kr/pcaps/mainpage                                                  | https://github.com/choilab/P-CAPS                                                                                 | 10.1089/cmb.2017.0066             | Journal of Computational Biology                                  | Prokaryotic Contig Annotation Pipeline Server: Web Application for a Prokaryotic Genome Annotation Pipeline Based on the Shiny App Package                                                               |
| 2017 | Pheasant Habitat Simulator                   | https://pheasant.shinyapps.io/pheasanthabitatsimulator/                                    | https://github.com/lsw5077/Pheasant-Habitat-Simulator                                                             | 10.1371/journal.pone.0188244      | Plos One                                                          | Translating statistical species-habitat models to interactive decision support tools                                                                                                                     |
| 2017 | PItcHPERFeCT                                 | http://johnmuschelli.com/ich_segment_all.html                                              | https://github.com/muschellij2/ichseg                                                                             | 10.1016/j.nicl.2017.02.007        | NeuroImage. Clinical.                                             | PItcHPERFeCT: Primary Intracranial Hemorrhage Probability Estimation using Random Forests on CT                                                                                                          |
| 2017 | REDCap                                       | NA                                                                                         | https://github.com/ul-research-support/spcapp                                                                     | 10.1016/j.ajic.2016.09.021        | American journal of infection control                             | Process control charts in infection prevention: Make it simple to make it happen                                                                                                                         |
| 2017 | repoDB                                       | http://apps.chiragjpgroup.org/repoDB/                                                      | https://github.com/adam-sam-brown/repoDB                                                                          | 10.1038/sdata.2017.29             | Scientific Data                                                   | A standard database for drug repositioning                                                                                                                                                               |
| 2017 | RRApp                                        | https://clinicalresearch-apps.com                                                          | NA                                                                                                                | 10.1017/cts.2017.310              | Journal of clinical and translational science                     | RRApp, a robust randomization app, for clinical and translational research                                                                                                                               |
| 2017 | ScanGEO                                      | http://scangeo.dartmouth.edu/ScanGEO/                                                      | https://github.com/StantonLabDartmouth/AppScanGEO                                                                 | 10.1093/bioinformatics/btx452     | Bioinformatics                                                    | ScanGEO: parallel mining of high-throughput gene expression data                                                                                                                                         |
| 2017 | ShinyFHIR                                    | NA                                                                                         | https://github.com/BD2KOnFHIR/ShinyFHIR                                                                           | 10.3233/978-1-61499-830-3-868     | Studies in Health Technology and Informatics                      | Shiny FHIR: An Integrated Framework Leveraging Shiny R and HL7 FHIR to Empower Standards-based Clinical Data Applications                                                                                |
| 2017 | ShinyGPAS                                    | https://chikudaisei.shinyapps.io/shinygpas/                                                | https://github.com/morota/ShinyGPAS                                                                               | 10.1186/s12711-017-0368-4         | Genetics, selection, evolution : GSE                              | ShinyGPAS: interactive genomic prediction accuracy simulator based on deterministic formulas                                                                                                             |
| 2017 | skelesim                                     | NA                                                                                         | https://github.com/christianparobek/skeleSim/blob/master/vignettes                                                | 10.1111/1755-0998.12607           | Molecular Ecology Resources                                       | skelesim: an extensible, general framework for population genetic simulation in R                                                                                                                        |
| 2017 | SpatialEpiApp                                | NA                                                                                         | https://github.com/Paula-Moraga/SpatialEpiApp                                                                     | 10.1016/j.sste.2017.08.001        | Spatial and Spatio-temporal Epidemiology                          | Spatial and Spatio-temporal Epidemiology                                                                                                                                                                 |
| 2017 | START                                        | https://kcvi.shinyapps.io/START/                                                           | https://github.com/jminnier/STARTapp                                                                              | 10.1093/bioinformatics/btw624     | Bioinformatics                                                    | The START App: a web-based RNAseq analysis and visualization resource                                                                                                                                    |
| 2017 | STOPGAP                                      | http://stopgapwebapp.com                                                                   | https://github.com/StatGenPRD/STOPGAP                                                                             | 10.1093/bioinformatics/btx274     | Bioinformatics                                                    | STOPGAP: a database for systematic target opportunity assessment by genetic association predictions                                                                                                      |
| 2017 | SubVis                                       | https://cran.r-project.org/web/packages/SubVis/                                            | https://github.com/sabarlowe/SubVis                                                                               | 10.7717/peerj.3492                | PeerJ                                                             | SubVis: an interactive R package for exploring the effects of multiple substitution matrices on pairwise sequence alignment                                                                              |
| 2017 | TSIS                                         | NA                                                                                         | https://github.com/wyguo/TSIS                                                                                     | 10.1093/bioinformatics/btx411     | Bioinformatics                                                    | TSIS: an R package to infer alternative splicing isoform switches for time-series data                                                                                                                   |
| 2017 | twoddpcr                                     | http://shiny.cruk.manchester.ac.uk/twoddpcr/                                               | https://bioconductor.org/packages/release/bioc/html/twoddpcr.html                                                 | 10.1093/bioinformatics/btx308     | Bioinformatics                                                    | twoddpcr: an R/Bioconductor package and Shiny app for Droplet Digital PCR analysis                                                                                                                       |
| 2017 | UpSetR                                       | https://gehlenborglab.shinyapps.io/upsetr/                                                 | https://github.com/hms-dbmi/UpSetR/                                                                               | 10.1093/bioinformatics/btx364     | Bioinformatics                                                    | UpSetR: an R package for the visualization of intersecting sets and their properties                                                                                                                     |
| 2018 | ANIMA                                        | https://zenodo.org/record/1163398                                                          | https://github.com/adeffur/anima                                                                                  | 10.12688/wellcomeopenres.14073.3  | Wellcome Open Research                                            | ANIMA: Association network integration for multiscale analysis                                                                                                                                           |
| 2018 | ASAR                                         | NA                                                                                         | https://github.com/Askarbek-orakov/ASAR                                                                           | 10.1093/bioinformatics/btx775     | Bioinformatics                                                    | ASAR: visual analysis of metagenomes in R                                                                                                                                                                |
| 2018 | BART                                         | http://igc1.salk.edu:3838/bart/                                                            | https://bitbucket.org/Luisa_amaral/bart                                                                           | 10.1186/s12859-018-2308-x         | BMC Bioinformatics                                                | BART: bioinformatics array research tool                                                                                                                                                                 |
| 2018 | BaySeqPeak                                   | https://qiwei.shinyapps.io/BaySeqPeak                                                      | https://github.com/liqiwei2000/BaySeqPeak                                                                         | 10.1007/s40484-018-0149-2         | Quantitative biology                                              | A Bayesian hierarchical model for analyzing methylated RNA immunoprecipitation sequencing data                                                                                                           |
| 2018 | Be-Breeder                                   | NA                                                                                         | http://www.genetica.esalq.usp.br/alogamas/R.html                                                                  | 10.1590/1984-70332018v18n2s36     | Crop Breeding and Applied Biotechnology                           | Be-Breeder: an R/Shiny application for phenotypic data analyses in plant breeding                                                                                                                        |
| 2018 | Bin-CE                                       | https://uesca-apps.shinyapps.io/bincep/                                                    | NA                                                                                                                | 10.1371/journal.pone.0209000      | Plos One                                                          | Bin-CE: A comprehensive web application to decide upon the best set of outcomes to be combined in a binary composite endpoint                                                                            |
| 2018 | BioInstaller                                 | NA                                                                                         | https://github.com/JhuangLab/BioInstaller                                                                         | 10.7717/peerj.5853                | PeerJ                                                             | BioInstaller: a comprehensive R package to construct interactive and reproducible biological data analysis applications based on the R platform                                                          |
| 2018 | BioJupies                                    | http://biojupies.cloud                                                                     | https://github.com/MaayanLab/biojupies                                                                            | 10.1016/j.cels.2018.10.007        | Cell Systems                                                      | BioJupies: Automated Generation of Interactive Notebooks for RNA-Seq Data Analysis in the Cloud                                                                                                          |
| 2018 | BLASTmap                                     | https://ics.hutton.ac.uk/blastmap/                                                         | https://github.com/kb-bioinf/BLASTmap                                                                             | 10.1007/978-1-4939-8724-5_14      | Methods in Molecular Biology                                      | BLASTmap: A Shiny-Based Application to Visualize BLAST Results as Interactive Heat Maps and a Tool to Design Gene-Specific Baits for Bespoke Target Enrichment Sequencing                                |
| 2018 | BRepertoire                                  | http://mabra.biomed.kcl.ac.uk/BRepertoire                                                  | NA                                                                                                                | 10.1093/nar/gky276                | Nucleic Acids Research                                            | BRepertoire: a user-friendly web server for analysing antibody repertoire data                                                                                                                           |
| 2018 | COEX-Seq                                     | NA                                                                                         | https://github.com/kimsc77/COEX-seq                                                                               | 10.5808/GI.2018.16.4.e36          | Genomics & Informatics                                            | COEX-Seq: Convert a Variety of Measurements of Gene Expression in RNA-Seq                                                                                                                                |
| 2018 | Coral                                        | http://phanstiel-lab.med.unc.edu/CORAL/                                                    | https://github.com/dphansti/Coral.html                                                                            | 10.1016/j.cels.2018.07.001        | Cell Systems                                                      | Coral: Clear and Customizable Visualization of Human Kinome Data                                                                                                                                         |
| 2018 | CRM                                          | https://uvatrapps.shinyapps.io/crmb/                                                       | http://faculty.virginia.edu/model-based_dose-finding/                                                             | 10.1186/s12885-018-4038-x         | BMC Cancer                                                        | A web tool for designing and conducting phase I trials using the continual reassessment method                                                                                                           |
| 2018 | DAME                                         | https://acnc-shinyapps.shinyapps.io/DAME/                                                  | https://github.com/bdpiccolo/ACNC-DAME                                                                            | 10.1093/bioinformatics/btx686     | Bioinformatics                                                    | Dynamic assessment of microbial ecology (DAME): a web app for interactive analysis and visualization of microbial sequencing data                                                                        |
| 2018 | ddPCRclust                                   | https://bibiserv.cebitec.uni-bielefeld.de/ddPCRvis/                                        | https://github.com/bgbrink/ddPCRvis/                                                                              | 10.1093/bioinformatics/bty136     | Bioinformatics                                                    | ddPCRclust: an R package and Shiny app for automated analysis of multiplexed ddPCR data                                                                                                                  |
| 2018 | DiNAR                                        | https://nib-si.shinyapps.io/DiNAR                                                          | https://github.com/NIB-SI/DiNAR                                                                                   | 10.1186/s13007-018-0345-0         | Plant Methods                                                     | DiNAR: revealing hidden patterns of plant signalling dynamics using Differential Network Analysis in R                                                                                                   |
| 2018 | EditR                                        | http://baseeditr.com/                                                                      | https://github.com/MoriarityLab/EditR7                                                                            | 10.1089/crispr.2018.0014          | The Crispr Journal                                                | A Method to Quantify Base Editing from Sanger Sequencing                                                                                                                                                 |
| 2018 | EFAshiny                                     | https://psychilin.shinyapps.io/EFAshiny/                                                   | https://github.com/PsyChiLin/EFAshiny                                                                             | 10.21105/joss.00567               | Journal of Open Source Software                                   | EFAshiny: An User-Friendly Shiny Application for Exploratory Factor Analysis                                                                                                                             |
| 2018 | Fractalis                                    | https://fractalis.lcsb.uni.lu/                                                             | NA                                                                                                                | 10.1093/gigascience/giy109        | Gigascience                                                       | Fractalis: a scalable open-source service for platform-independent interactive visual analysis of biomedical data                                                                                        |
| 2018 | funRiceGenes                                 | https://venyao.xyz/RICENCODE/                                                              | https://github.com/venyao/RICENCODE                                                                               | 10.1093/gigascience/gix119        | GigaScience                                                       | funRiceGenes dataset for comprehensive understanding and application of rice functional genes                                                                                                            |
| 2018 | Gazepath                                     | NA                                                                                         | https://github.com/cran/gazepath                                                                                  | 10.3758/s13428-017-0909-3         | Behavior Research Methods                                         | Gazepath: An eye-tracking analysis tool that accounts for individual differences and data quality                                                                                                        |
| 2018 | GDCRNATools                                  | NA                                                                                         | https://github.com/Jialab-UCR/GDCRNATools                                                                         | 10.1093/bioinformatics/bty124     | Bioinformatics                                                    | GDCRNATools: an R/Bioconductor package for integrative analysis of lncRNA, miRNA and mRNA data in GDC                                                                                                    |
| 2018 | GEE Sample Size Calculator                   | https://djprice.shinyapps.io/gee_calculator/                                               | NA                                                                                                                | 10.1111/ppe.12471                 | Paediatric and perinatal epidemiology                             | Accounting for twin births in sample size calculations for randomised trials                                                                                                                             |
| 2018 | gQTL                                         | https://genomics.tamu.edu/gqtl                                                             | https://github.com/dmgatti/DOQTL                                                                                  | 10.1534/g3.118.200230             | G3 (Bethesda, Md.)                                                | gQTL: A Web Application for QTL Analysis Using the Collaborative Cross Mouse Genetic Reference Population                                                                                                |
| 2018 | HEAT Plus                                    | https://www.gnu.org/licenses/gpl-2.0                                                       | https://github.com/WHOequity/HEATPlus-1.0                                                                         | 10.1080/16549716.2018.1440783     | Global Health Action                                              | Health Equity Assessment Toolkit Plus (HEAT Plus): software for exploring and comparing health inequalities using uploaded datasets                                                                      |
| 2018 | HTPmod                                       | http://www.epiplant.hu-berlin.de/shiny/app/HTPmod/                                         | https://github.com/htpmod/HTPmod-shinyApp                                                                         | 10.1038/s42003-018-0091-x         | Communications Biology                                            | The HTPmod Shiny application enables modeling and visualization of large-scale biological data                                                                                                           |
| 2018 | iDEP                                         | http://bioinformatics.sdstate.edu/idep/                                                    | https://github.com/iDEP-SDSU/idep/                                                                                | 10.1186/s12859-018-2486-6         | BMC Bioinformatics                                                | iDEP: an integrated web application for differential expression and pathway analysis of RNA-Seq data                                                                                                     |
| 2018 | iDINGO                                       | NA                                                                                         | https://github.com/MinJinHa/iDINGO                                                                                | 10.1093/bioinformatics/btx750     | Bioinformatics                                                    | iDINGO-integrative differential network analysis in genomics with Shiny application                                                                                                                      |
| 2018 | IDO-laad                                     | https://www.idolaad.com/                                                                   | NA                                                                                                                | 10.3390/wevj9020025               | World Electric Vehicle Journal                                    | Performance of Electric Vehicle Charging Infrastructure: Development of an Assessment Platform Based on Charging Data                                                                                    |
| 2018 | INfORM                                       | NA                                                                                         | https://github.com/Greco-Lab/INfORM                                                                               | 10.1093/bioinformatics/bty063     | Bioinformatics                                                    | INfORM: Inference of NetwOrk Response Modules                                                                                                                                                            |
| 2018 | IntLIM                                       | NA                                                                                         | https://github.com/mathelab/IntLIM                                                                                | 10.1186/s12859-018-2085-6         | BMC Bioinformatics                                                | IntLIM: integration using linear models of metabolomics and gene expression data                                                                                                                         |
| 2018 | Ioncopy                                      | NA                                                                                         | https://cran.r-project.org/web/packages/ioncopy/index.html                                                        | 10.1186/s12859-018-2159-5         | BMC Bioinformatics                                                | Ioncopy: an R Shiny app to call copy number alterations in targeted NGS data                                                                                                                             |
| 2018 | iS-CellR                                     | NA                                                                                         | https://github.com/immcore/iS-CellR                                                                               | 10.1093/bioinformatics/bty517     | Bioinformatics                                                    | iS-CellR: a user-friendly tool for analyzing and visualizing single-cell RNA sequencing data                                                                                                             |
| 2018 | iSeq                                         | http://iseq.cbi.pku.edu.cn                                                                 | https://github.com/ChengLiLab/iSeq                                                                                | 10.1007/978-1-4939-7717-8_10      | Methods in Molecular Biology                                      | iSeq: Web-Based RNA-seq Data Analysis and Visualization                                                                                                                                                  |
| 2018 | IsoFishR                                     | NA                                                                                         | https://github.com/MalteWillmes/IsoFishR                                                                          | 10.1371/journal.pone.0204519      | Plos One                                                          | IsoFishR: An application for reproducible data reduction and analysis of strontium isotope ratios (87Sr/86Sr) obtained via laser-ablation MC-ICP-MS                                                      |
| 2018 | KLK-CANMAP                                   | https://cancerbioinformatics.shinyapps.io/klk-canmap/                                      | NA                                                                                                                | 10.1515/hsz-2017-0322             | Biological Chemistry                                              | Mining human cancer datasets for kallikrein expression in cancer: the 'KLK-CANMAP' Shiny web tool                                                                                                        |
| 2018 | LOLAweb                                      | http://lolaweb.databio.org                                                                 | http://github.com/databio/LOLAweb                                                                                 | 10.1093/nar/gky464                | Nucleic Acids Research                                            | LOLAweb: a containerized web server for interactive genomic locus overlap enrichment analysis                                                                                                            |
| 2018 | MetaboCraft                                  | https://hub.docker.com/r/ronandaly/metabocraft/                                            | https://github.com/argymeg/MetaboCraft/                                                                           | 10.1093/bioinformatics/bty102     | Bioinformatics                                                    | MetaboCraft: building a Minecraft plugin for metabolomics                                                                                                                                                |
| 2018 | Metaviz                                      | http://www.metaviz.org                                                                     | https://github.com/epiviz/metaviz-docker]                                                                         | 10.1093/nar/gky136                | Nucleic Acids Research                                            | Metaviz: interactive statistical and visual analysis of metagenomic data                                                                                                                                 |
| 2018 | methylGSA                                    | http://ec2-35-174-138-147.compute-1.amazonaws.com:3838/methylGSA/                          | http://www.ams.sunysb.edu/~pfkuan/softwares.html#methylGSA                                                        | 10.1093/bioinformatics/bty892     | Bioinformatics                                                    | methylGSA: a Bioconductor package and Shiny app for DNA methylation data length bias adjustment in gene set testing                                                                                      |
| 2018 | MicrobiomeDB                                 | http://microbiomeDB.org                                                                    | https://github.com/dpbisme/microbiomeDB                                                                           | 10.1093/nar/gkx1027               | Nucleic Acids Research                                            | MicrobiomeDB: a systems biology platform for integrating, mining and analyzing microbiome experiments                                                                                                    |
| 2018 | miRBaseConverter                             | http://bioconductor.org/packages/miRBaseConverter/                                         | https://taoshengxu.github.io/ExperimentData/miRBaseConverter                                                      | 10.1186/s12859-018-2531-5         | BMC Bioinformatics                                                | miRBaseConverter: an R/Bioconductor package for converting and retrieving miRNA name, accession, sequence and family information in different versions of miRBase                                        |
| 2018 | miRCancerdb                                  | https://mahshaaban.shinyapps.io/miRCancerdb/                                               | https://github.com/MahShaaban/miRCancerdb                                                                         | 10.1186/s13104-018-3160-9         | BMC Research Notes                                                | miRCancerdb: a database for correlation analysis between microRNA and gene expression in cancer                                                                                                          |
| 2018 | miRTissue                                    | http://tblab.pa.icar.cnr.it/mirtissue.html                                                 | NA                                                                                                                | 10.1186/s12859-018-2418-5         | BMC Bioinformatics                                                | miRTissue: a web application for the analysis of miRNA-target interactions in human tissues                                                                                                              |
| 2018 | MSDshiny                                     | https://ph-shiny.iowa.uiowa.edu/rpterson/MSDshiny/                                         | https://github.com/petersonR/MSDshiny                                                                             | 10.1177/1740774518789098          | Clinical Trials                                                   | Application of multi-state models in cancer clinical trials                                                                                                                                              |
| 2018 | MuSiCa                                       | http://bioinfo.ciberehd.org/GPtoCRC/en/tools.html                                          | https://github.com/marcos-diazg/musica                                                                            | 10.1186/s12859-018-2234-y         | BMC Bioinformatics                                                | Mutational Signatures in Cancer (MuSiCa): a web application to implement mutational signatures analysis in cancer samples                                                                                |
| 2018 | Netpredictor                                 | https://github.com/abhik1368/ShinyNetPredictor                                             | https://github.com/abhik1368/netpredictor                                                                         | 10.1186/s12859-018-2254-7         | BMC Bioinformatics                                                | Netpredictor: R and Shiny package to perform drug-target network analysis and prediction of missing links                                                                                                |
| 2018 | PAGEANT                                      | https://andrewhaoyu.shinyapps.io/PAGEANT/                                                  | NA                                                                                                                | 10.1093/bioinformatics/btx770     | Bioinformatics                                                    | Power Analysis for Genetic Association Test (PAGEANT) provides insights to challenges for rare variant association studies                                                                               |
| 2018 | PIVOT                                        | http://kim.bio.upenn.edu/software/pivot.shtml                                              | http://kim.bio.upenn.edu/software/pivot.shtml                                                                     | 10.1186/s12859-017-1994-0         | BMC Bioinformatics                                                | PIVOT: platform for interactive analysis and visualization of transcriptomics data                                                                                                                       |
| 2018 | RaMP                                         | https://ramp-db.bmi.osumc.edu/                                                             | https://github.com/Mathelab/RaMP-DB                                                                               | 10.3390/metabo8010016             | Metabolites                                                       | RaMP: A Comprehensive Relational Database of Metabolomics Pathways for Pathway Enrichment Analysis of Genes and Metabolites                                                                              |
| 2018 | ranacapa                                     | https://gauravsk.shinyapps.io/ranacapa/                                                    | https://github.com/gauravsk/ranacapa                                                                              | 10.12688/f1000research.16680.1    | F1000Research                                                     | ranacapa: An R package and Shiny web app to explore environmental DNA data with exploratory statistics and interactive visualizations                                                                    |
| 2018 | regQTLs                                      | NA                                                                                         | https://github.com/gawilk/mirApp                                                                                  | 10.1371/journal.pgen.1007837      | Plos Genetics                                                     | regQTLs: Single nucleotide polymorphisms that modulate microRNA regulation of gene expression in tumors                                                                                                  |
| 2018 | RIVET                                        | https://ruggleslab.shinyapps.io/RIVET/                                                     | NA                                                                                                                | 10.1186/s12864-018-5166-z         | BMC Genomics                                                      | RIVET: comprehensive graphic user interface for analysis and exploration of genome-wide translatomics data                                                                                               |
| 2018 | SeesawPred                                   | http://seesaw.lcsb.uni.lu                                                                  | NA                                                                                                                | 10.1038/s41598-018-31688-9        | Scientific Reports                                                | SeesawPred: A Web Application for Predicting Cell-fate Determinants in Cell Differentiation                                                                                                              |
| 2018 | Shiny-AESC                                   | https://dczhang.shinyapps.io/expectancyApp/                                                | https://github.com/dczhang1/ShinyAESC                                                                             | 10.3389/fpsyg.2018.01221          | Frontiers in psychology                                           | Utility of Alternative Effect Size Statistics and the Development of a Web-Based Calculator: Shiny-AESC                                                                                                  |
| 2018 | ShinyAIM                                     | https://chikudaisei.shinyapps.io/shinyaim/                                                 | https://github.com/whussain2/ShinyAIM                                                                             | 10.1002/pld3.91                   | Plant Direct                                                      | ShinyAIM: Shiny‐based application of interactive Manhattan plots for longitudinal genome‐wide association studies                                                                                        |
| 2018 | shinyChromosome                              | https://venyao.xyz/shinyChromosome/                                                        | https://github.com/venyao/shinyChromosome                                                                         | 10.1016/j.gpb.2019.07.003         | Genomics, Proteomics & Bioinformatics                             | shinyChromosome: an R/Shiny application for interactive creation of non-circular plots of whole genomes                                                                                                  |
| 2018 | shinyCircos                                  | https://venyao.xyz/shinyCircos/                                                            | https://github.com/venyao/shinyCircos                                                                             | 10.1093/bioinformatics/btx763     | Bioinformatics                                                    | shinyCircos: an R/Shiny application for interactive creation of Circos plot                                                                                                                              |
| 2018 | ShinyCNV                                     | NA                                                                                         | https://github.com/gzhmat/ShinyCNV                                                                                | 10.1093/bioinformatics/bty546     | Bioinformatics                                                    | ShinyCNV: a Shiny/R application to view and annotate DNA copy number variations                                                                                                                          |
| 2018 | ShinyCortex                                  | https://bioinf.eva.mpg.de/shiny/sample-apps/ShinyCortex/                                   | NA                                                                                                                | 10.3389/fnins.2018.00315          | Frontiers in Neuroscience                                         | ShinyCortex: Exploring Single-Cell Transcriptome Data From the Developing Human Cortex                                                                                                                   |
| 2018 | shinyGISPA                                   | http://shinygispa.winship.emory.edu/shinyGISPA/                                            | http://bioconductor.org/packages/GISPA/                                                                           | 10.1371/journal.pone.0192563      | Plos One                                                          | shinyGISPA: A web application for characterizing phenotype by gene sets using multiple omics data combinations                                                                                           |
| 2018 | shinySISPA                                   | http://bbisr.shinyapps.winship.emory.edu/shinySISPA/                                       | https://github.com/BhaktiDwivedi/shinySISPA                                                                       | 10.12688/f1000research.13934.2    | F1000Research                                                     | shinySISPA: A web tool for defining sample groups using gene sets from multiple-omics data                                                                                                               |
| 2018 | SimExTargId                                  | NA                                                                                         | https://github.com/JosieLHayes/simExTargId                                                                        | 10.1093/bioinformatics/bty218     | Bioinformatics                                                    | SimExTargId: a comprehensive package for real-time LC-MS data acquisition and analysis                                                                                                                   |
| 2018 | SLiM-Enrich                                  | http://shiny.slimsuite.unsw.edu.au/SLiMEnrich/                                             | https://github.com/slimsuite/SLiMEnrich                                                                           | 10.7717/peerj.5858                | PeerJ                                                             | SLiM-Enrich: computational assessment of protein-protein interaction data as a source of domain-motif interactions                                                                                       |
| 2018 | SRGE                                         | https://tavoun.shinyapps.io/SRGE/                                                          | NA                                                                                                                | 10.1097/OLQ.0000000000000699      | Sexually transmitted diseases                                     | A Web Application to Facilitate Syphilis Reactor Grid Evaluations                                                                                                                                        |
| 2018 | STADIUM                                      | http://stadium.pmrc.re.kr                                                                  | NA                                                                                                                | 10.5808/GI.2018.16.4.e28          | Genomics & Informatics                                            | STADIUM: Species-Specific tRNA Adaptive Index Compendium                                                                                                                                                 |
| 2018 | TextGridShiny                                | https://sudoranais.shinyapps.io/Analysis_Processing_Rhotic_Alveolar/                       | https://github.com/buhem/TextGridShiny                                                                            | 10.5281/zenodo.3364654            | Zenodo                                                            | TextGridShiny - Create interactive annotation interfaces for speech with R and Shiny                                                                                                                     |
| 2018 | Trendy                                       | https://bioconductor.org/packages/release/bioc/html/Trendy.html                            | https://github.com/rhondabacher/Trendy                                                                            | 10.1186/s12859-018-2405-x         | BMC Bioinformatics                                                | Trendy: segmented regression analysis of expression dynamics in high-throughput ordered profiling experiments                                                                                            |
| 2018 | TRGAted                                      | http://tcpaportal.org/tcpa/download.html                                                   | https://github.com/ncborcherding/TRGAted                                                                          | 10.12688/f1000research.15789.2    | F1000Research                                                     | TRGAted: A web tool for survival analysis using protein data in the Cancer Genome Atlas                                                                                                                  |
| 2018 | TumGrowth                                    | https://kroemerlab.shinyapps.io/TumGrowth/                                                 | NA                                                                                                                | 10.1080/2162402X.2018.1462431     | Oncoimmunology                                                    | TumGrowth: An open-access web tool for the statistical analysis of tumor growth curves                                                                                                                   |
| 2018 | TVT                                          | https://elk.med.harvard.edu/gfy/www/modules/single_standard/                               | NA                                                                                                                | 10.1021/acs.jproteome.8b00737     | Journal of Proteome Research                                      | Web-Based Search Tool for Visualizing Instrument Performance Using the Triple Knockout (TKO) Proteome Standard                                                                                           |
| 2018 | unequal-allocation                           | https://artool.shinyapps.io/unequal-allocation/                                            | https://github.com/yevgenryeznik/restricted                                                                       | 10.1002/sim.7817                  | Statistics in medicine                                            | A comparative study of restricted randomization procedures for multiarm trials with equal or unequal treatment allocation ratios                                                                         |
| 2018 | VAReporter                                   | http://rnd.cgu.edu.tw/vareporter                                                           | NA                                                                                                                | 10.1186/s12864-018-4468-5         | BMC Genomics                                                      | VAReporter: variant reporter for cancer research of massive parallel sequencing                                                                                                                          |
| 2018 | Viridot                                      | NA                                                                                         | https://github.com/leahkatzelnick/Viridot                                                                         | 10.1371/journal.pntd.0006862      | Plos Neglected Tropical Diseases                                  | Viridot: An automated virus plaque (immunofocus) counter for the measurement of serological neutralizing responses with application to dengue virus                                                      |
| 2018 | WEADE                                        | http://beta-weade.cos.uni-heidelberg.de                                                    | https://bitbucket.org/LetiP/weade                                                                                 | 10.1371/journal.pone.0204016      | Plos One                                                          | WEADE: A workflow for enrichment analysis and data exploration                                                                                                                                           |
| 2018 | WebSurvCa                                    | https://shiny.snpstats.net/WebSurvCa/                                                      | NA                                                                                                                | 10.1016/j.gaceta.2017.10.015      | Gaceta sanitaria                                                  | [WebSurvCa: web-based estimation of death and survival probabilities in a cohort]                                                                                                                        |
| 2018 | WHAM                                         | https://ruggleslab.shinyapps.io/wham_v1/                                                   | https://github.com/ruggleslab/jukebox/tree/master/wham_v1                                                         | 10.1186/s12864-018-4870-z         | BMC Genomics                                                      | WHAM!: a web-based visualization suite for user-defined analysis of metagenomic shotgun sequencing data                                                                                                  |
| 2018 | WSSPAS                                       | http://biostatapps.inonu.edu.tr/WSSPAS                                                     | NA                                                                                                                | 10.5336/biostatic.2018-62787      | Turkiye Klinikleri Journal of Biostatistics                       | WSSPAS: An Interactive Web Application for Sample Size and Power Analysis with R Using Shiny                                                                                                             |
| 2019 | Adjutant                                     | NA                                                                                         | https://github.com/amcrisan/Adjutant                                                                              | 10.1093/bioinformatics/bty722     | Bioinformatics                                                    | Adjutant: an R-based tool to support topic discovery for systematic and literature reviews                                                                                                               |
| 2019 | Allocation-Power-Optimizer                   | NA                                                                                         | https://github.com/stefangraw/Allocation-Power-Optimizer                                                          | 10.1080/00031305.2019.1610065     | The American statistician                                         | Optimizing Sample Size Allocation and Power in a Bayesian Two-Stage Drop-The-Losers Design                                                                                                               |
| 2019 | Anabel                                       | www.skscience.org/anabel                                                                   | https://github.com/SKscience/Anabel                                                                               | 10.1177/1177932218821383          | Bioinformatics and Biology Insights                               | Anabel: An Online Tool for the Real-Time Kinetic Analysis of Binding Events                                                                                                                              |
| 2019 | Aneuvis                                      | https://dpique.shinyapps.io/aneuvis/                                                       | https://github.com/dpique/aneuvis                                                                                 | 10.1186/s12859-019-2842-1         | BMC Bioinformatics                                                | Aneuvis: web-based exploration of numerical chromosomal variation in single cells                                                                                                                        |
| 2019 | ARMOR                                        | NA                                                                                         | https://github.com/csoneson/ARMOR                                                                                 | 10.1534/g3.119.400185             | G3 (Bethesda, Md.)                                                | ARMOR: An Automated Reproducible MOdular Workflow for Preprocessing and Differential Analysis of RNA-seq Data                                                                                            |
| 2019 | bcROCsurface                                 | http://khanhtoduc.shinyapps.io/bcROCsurface_shiny/                                         | http://CRAN.R-project.org/package=bcROCsurface                                                                    | 10.1186/s12859-017-1914-3         | BMC Bioinformatics                                                | bcROCsurface: an R package for correcting verification bias in estimation of the ROC surface and its volume for continuous diagnostic tests                                                              |
| 2019 | BISR-RNAseq                                  | https://dataportal.bmi.osumc.edu/RNA_Seq/                                                  | https://code.bmi.osumc.edu/gadepalli.3/BISR-RNAseq-ICIBM2019                                                      | 10.1186/s12859-019-3251-1         | BMC Bioinformatics                                                | BISR-RNAseq: an efficient and scalable RNAseq analysis workflow with interactive report generation                                                                                                       |
| 2019 | BOM                                          | NA                                                                                         | https://github.com/fgabriel1891/BiodiversityObservationsMiner                                                     | 10.3897/BDJ.7.e28737              | Biodiversity Data Journal                                         | Biodiversity Observations Miner: A web application to unlock primary biodiversity data from published literature                                                                                         |
| 2019 | CANDI                                        | https://candi.nextgenhealthcare.org/                                                       | https://github.com/mbadge/candi                                                                                   | 10.1093/bioinformatics/bty855     | Bioinformatics                                                    | CANDI: an R package and Shiny app for annotating radiographs and evaluating computer-aided diagnosis                                                                                                     |
| 2019 | CarboLogR                                    | NA                                                                                         | https://github.com/kevinVervier/CarboLogR                                                                         | 10.1101/695676                    | bioRxiv                                                           | CarboLogR: a Shiny/R application for statistical analysis of bacterial utilisation of carbon sources                                                                                                     |
| 2019 | CardiacPBPK                                  | NA                                                                                         | https://github.com/jszlek/CardiacPBPK                                                                             | 10.1016/j.compbiomed.2019.103484  | Computers in Biology and Medicine                                 | CardiacPBPK: A tool for the prediction and visualization of time-concentration profiles of drugs in heart tissue                                                                                         |
| 2019 | CCDBcurator and EyeChrom                     | http://eyechrom.com                                                                        | https://github.com/RodrigoRivero/EyeChrom                                                                         | 10.1002/aps3.1207                 | Applications in Plant Sciences                                    | EyeChrom and CCDBcurator: Visualizing chromosome count data from plants                                                                                                                                  |
| 2019 | checkMyIndex                                 | https://checkmyindex.pasteur.fr/                                                           | https://github.com/PF2-pasteur-fr/checkMyIndex                                                                    | 10.1093/bioinformatics/bty706     | Bioinformatics                                                    | checkMyIndex: a web-based R/Shiny interface for choosing compatible sequencing indexes                                                                                                                   |
| 2019 | Chordomics                                   | https://kmcd.shinyapps.io/chordomics/                                                      | https://github.com/kevinmcdonnell6/chordomics                                                                     | 10.1093/bioinformatics/btz711     | Bioinformatics                                                    | Chordomics: a visualisation tool for linking function to phylogeny in microbiomes                                                                                                                        |
| 2019 | CNspector                                    | https://shiny.wehi.edu.au/jmarkham/CNspector/index.html                                    | https://github.com/PapenfussLab/Cnspector                                                                         | 10.1038/s41598-019-42858-8        | Scientific Reports                                                | CNspector: a web-based tool for visualisation and clinical diagnosis of copy number variation from next generation sequencing                                                                            |
| 2019 | Colliders                                    | http://watzilei.com/shiny/collider/                                                        | https://github.com/migariane/ColliderApp                                                                          | 10.1093/ije/dyy275                | International Journal of Epidemiology                             | Educational Note: Paradoxical collider effect in the analysis of non-communicable disease epidemiological data: a reproducible illustration and web application                                          |
| 2019 | colocr                                       | https://mahshaaban.shinyapps.io/colocr_app2/                                               | https://github.com/ropensci/colocr                                                                                | 10.7717/peerj.7255                | PeerJ                                                             | colocr: an R package for conducting co-localization analysis on fluorescence microscopy images                                                                                                           |
| 2019 | DEBrowser                                    | https://bioconductor.org/packages/release/bioc/html/debrowser.html                         | https://github.com/UMMS-Biocore/debrowser                                                                         | 10.1186/s12864-018-5362-x         | BMC Genomics                                                      | DEBrowser: interactive differential expression analysis and visualization tool for count data                                                                                                            |
| 2019 | DiffNetFDR                                   | NA                                                                                         | https://github.com/Zhangxf-ccnu/DiffNetFDR                                                                        | 10.1093/bioinformatics/btz051     | Bioinformatics                                                    | DiffNetFDR: differential network analysis with false discovery rate control                                                                                                                              |
| 2019 | DNABarcodeCompatibility                      | https://dnabarcodecompatibility.pasteur.fr                                                 | https://github.com/comoto-pasteur-fr                                                                              | 10.1093/bioinformatics/bty1030    | Bioinformatics                                                    | DNABarcodeCompatibility: an R-package for optimizing DNA-barcode combinations in multiplex sequencing experiments                                                                                        |
| 2019 | ECOGEMS                                      | https://venyao.xyz/ECOGEMS/                                                                | https://github.com/venyao/ECOGEMS                                                                                 | 10.1093/bioinformatics/btz186     | Bioinformatics                                                    | ECOGEMS: efficient compression and retrieve of SNP data of 2058 rice accessions with integer sparse matrices                                                                                             |
| 2019 | EDGE in TCGA                                 | http://ls-shiny-prod.uwm.edu/edge_in_tcga                                                  | https://github.com/andreamrau/EDGE-in-TCGA                                                                        | 10.1093/bioinformatics/bty551     | Bioinformatics                                                    | Exploring drivers of gene expression in the Cancer Genome Atlas                                                                                                                                          |
| 2019 | eDNA                                         | https://seak.shinyapps.io/eDNA/                                                            | NA                                                                                                                | 10.1101/2020.12.09.417600         | bioRxiv                                                           | An Rshiny app for modelling environmental DNA data: accounting for false positive and false negative observation error                                                                                   |
| 2019 | EiaD                                         | https://eyeintegration.nei.nih.gov/                                                        | https://github.com/davemcg/eyeIntegration_app/                                                                    | 10.1167/iovs.19-27106             | Investigative Ophthalmology & Visual Science                      | Eye in a Disk: eyeIntegration Human Pan-Eye and Body Transcriptome Database Version 1.0                                                                                                                  |
| 2019 | ENCORE                                       | NA                                                                                         | https://github.com/delosh653/ENCORE                                                                               | 10.1145/3307339.3342137           | ACM BCB                                                           | ENCORE: A Visualization Tool for Insight into Circadian Omics                                                                                                                                            |
| 2019 | EndoDB                                       | https://vibcancer.be/software-tools/endodb                                                 | NA                                                                                                                | 10.1093/nar/gky997                | Nucleic Acids Research                                            | EndoDB: a database of endothelial cell transcriptomics data                                                                                                                                              |
| 2019 | EnImpute                                     | https://github.com/Zhangxf-ccnu/EnImpute                                                   | https://github.com/Zhangxf-ccnu/EnImpute                                                                          | 10.1093/bioinformatics/btz435     | Bioinformatics                                                    | EnImpute: imputing dropout events in single cell RNA sequencing data via ensemble learning                                                                                                               |
| 2019 | EpiExploreR                                  | https://pub.epiexplorer.izs.it/                                                            | NA                                                                                                                | 10.3390/microorganisms7120680     | Microorganisms                                                    | EpiExploreR: A Shiny Web Application for the Analysis of Animal Disease Data                                                                                                                             |
| 2019 | Epiviz2                                      | http://epiviz2.cbcb.umd.edu/                                                               | https://github.com/epiviz/epivizrChart                                                                            | 10.12688/f1000research.15433.1    | F1000Research                                                     | Epiviz Web Components: reusable and extensible component library to visualize functional genomic datasets                                                                                                |
| 2019 | eUTOPIA                                      | NA                                                                                         | https://github.com/Greco-Lab/eUTOPIA                                                                              | 10.1186/s13029-019-0071-7         | Source Code for Biology and Medicine                              | eUTOPIA: solUTion for Omics data PreprocessIng and Analysis                                                                                                                                              |
| 2019 | FastqCleaner                                 | NA                                                                                         | https://github.com/leandroroser/FastqCleaner                                                                      | 10.1186/s12859-019-2961-8         | BMC Bioinformatics                                                | FastqCleaner: an interactive Bioconductor application for quality-control, filtering and trimming of FASTQ files                                                                                         |
| 2019 | FunMappOne                                   | NA                                                                                         | https://github.com/grecolab/FunMappOne                                                                            | 10.1186/s12859-019-2639-2         | BMC Bioinformatics                                                | FunMappOne: a tool to hierarchically organize and visually navigate functional gene annotations in multiple experiments                                                                                  |
| 2019 | G4HunterApps                                 | NA                                                                                         | https://github.com/LacroixLaurent                                                                                 | 10.1093/bioinformatics/bty951     | Bioinformatics                                                    | G4HunterApps                                                                                                                                                                                             |
| 2019 | GENAVi                                       | https://junkdnalab.shinyapps.io/GENAVi/                                                    | https://github.com/alpreyes/GENAVi                                                                                | 10.1186/s12864-019-6073-7         | BMC Genomics                                                      | GENAVi: a shiny web application for gene expression normalization, analysis and visualization                                                                                                            |
| 2019 | GenomeGraphR                                 | https://fda-riskmodels.foodrisk.org/genomegraphr/                                          | NA                                                                                                                | 10.1371/journal.pone.0213039      | Plos One                                                          | GenomeGraphR: A user-friendly open-source web application for foodborne pathogen whole genome sequencing data integration, analysis, and visualization                                                   |
| 2019 | GNSplex                                      | NA                                                                                         | https://github.com/NPhogat/GNSplex                                                                                | 10.1007/s00604-018-3195-6         | Microchimica Acta                                                 | A smartphone readout system for gold nanoparticle-based lateral flow assays: application to monitoring of digoxigenin                                                                                    |
| 2019 | GREIN                                        | http://www.ilincs.org/apps/grein/                                                          | https://github.com/uc-bd2k/grein                                                                                  | 10.1038/s41598-019-43935-8        | Scientific Reports                                                | GREIN: An Interactive Web Platform for Re-analyzing GEO RNA-seq Data                                                                                                                                     |
| 2019 | GScluster                                    | http://gsaseq.appex.kr                                                                     | https://github.com/unistbig/GScluster                                                                             | 10.1186/s12864-019-5738-6         | BMC Genomics                                                      | GScluster: network-weighted gene-set clustering analysis                                                                                                                                                 |
| 2019 | GSS                                          | www.genesculpt.org                                                                         | https://github.com/Dobbs-Lab/GTagHD, https://github.com/Dobbs-Lab/MEDJED, and https://github.com/Dobbs-Lab/MENTHU | 10.1093/nar/gkz405                | Nucleic Acids Research                                            | The Gene Sculpt Suite: a set of tools for genome editing                                                                                                                                                 |
| 2019 | GWEHS                                        | http://gwehs.uvigo.es                                                                      | https://gitlab.com/elcortegano/GWEHS                                                                              | 10.3390/genes10080558             | Genes                                                             | GWEHS: A Genome-Wide Effect Sizes and Heritability Screener                                                                                                                                              |
| 2019 | hfs_app                                      | https://pkpdia.shinyapps.io/hfs_app/                                                       | NA                                                                                                                | 10.1038/s41598-019-49556-5        | Scientific Reports                                                | Mimicking in-vivo exposures to drug combinations in-vitro: anti-tuberculosis drugs in lung lesions and the hollow fiber model of infection                                                               |
| 2019 | HiCeekR                                      | http://bioinfo.na.iac.cnr.it/hiceekr/                                                      | https://github.com/lucidif/HiCeekR                                                                                | 10.3389/fgene.2019.01079          | Frontiers in Genetics                                             | HiCeekR: A Novel Shiny App for Hi-C Data Analysis                                                                                                                                                        |
| 2019 | HospMap                                      | https://cathsmith57.shinyapps.io/HospMap/                                                  | https://github.com/cathsmith57/HospMap                                                                            | 10.12688/wellcomeopenres.15240.2  | Wellcome Open Research                                            | An interactive data visualisation application to investigate nosocomial transmission of infections                                                                                                       |
| 2019 | IC-RISC                                      | https://ic-risc.esocan.org                                                                 | https://github.com/FredHutch/IC-RISC-Working                                                                      | 10.1186/s12876-019-1022-0         | BMC Gastroenterology                                              | Interactive decision support for esophageal adenocarcinoma screening and surveillance                                                                                                                    |
| 2019 | IRESpy                                       | https://irespy.shinyapps.io/IRESpy/                                                        | NA                                                                                                                | 10.1186/s12859-019-2999-7         | BMC Bioinformatics                                                | IRESpy: an XGBoost model for prediction of internal ribosome entry sites                                                                                                                                 |
| 2019 | IRIS-EDA                                     | http://bmbl.sdstate.edu/IRIS/                                                              | https://github.com/btmonier/iris                                                                                  | 10.1371/journal.pcbi.1006792      | PLoS Computational Biology                                        | IRIS-EDA: An integrated RNA-Seq interpretation system for gene expression data analysis                                                                                                                  |
| 2019 | iSwathX                                      | https://biolinfo.shinyapps.io/iSwathX/                                                     | https://github.com/znoor/iSwathX                                                                                  | 10.1093/bioinformatics/bty660     | Bioinformatics                                                    | iSwathX: an interactive web-based application for extension of DIA peptide reference libraries                                                                                                           |
| 2019 | iTUPA                                        | http://nuvem.ufabc.edu.br/itupa                                                            | NA                                                                                                                | 10.1093/bioinformatics/btz487     | Bioinformatics                                                    | iTUPA: an online automated application to perform Topographic-Unit Parsimony Analysis                                                                                                                    |
| 2019 | KRAB ZNF explorer                            | http://mi2.mini.pw.edu.pl:8080/KRAB_ZNF/                                                   | https://github.com/MI2DataLab/KRAB_ZNF/tree/master/app                                                            | 10.1093/bioinformatics/btz624     | Bioinformatics                                                    | KRAB ZNF explorer - the online tool for the exploration of the transcriptomic profiles of KRAB-ZNF factors in The Cancer Genome Atlas                                                                    |
| 2019 | learnPopGen                                  | NA                                                                                         | https://github.com/liamrevell/PopGen.apps                                                                         | 10.1002/ece3.5412                 | Ecology and Evolution                                             | learnPopGen: An R package for population genetic simulation and numerical analysis.                                                                                                                      |
| 2019 | LION/web                                     | NA                                                                                         | https://github.com/martijnmolenaar/LION-web/                                                                      | 10.1093/gigascience/giz061        | GigaScience                                                       | LION/web: a web-based ontology enrichment tool for lipidomic data analysis                                                                                                                               |
| 2019 | Lipid Mini-On                                | https://omicstools.pnnl.gov/shiny/lipid-mini-on/                                           | https://github.com/PNNL-Comp-Mass-Spec/LipidMiniOn                                                                | 10.1093/bioinformatics/btz250     | Bioinformatics                                                    | Lipid Mini-On: mining and ontology tool for enrichment analysis of lipidomic data                                                                                                                        |
| 2019 | MaGIC                                        | NA                                                                                         | https://github.com/gimelbrantlab/magic.                                                                           | 10.1186/s12859-019-2679-7         | BMC Bioinformatics                                                | MaGIC: a machine learning tool set and web application for monoallelic gene inference from chromatin                                                                                                     |
| 2019 | MaizeSNPDB                                   | https://venyao.xyz/MaizeSNPDB/                                                             | https://github.com/venyao/MaizeSNPDB                                                                              | 10.1016/j.csbj.2019.10.003        | Computational and Structural Biotechnology Journal                | MaizeSNPDB: A comprehensive database for efficient retrieve and analysis of SNPs among 1210 maize lines                                                                                                  |
| 2019 | Mayo Clinic                                  | https://rtools.mayo.edu/home/                                                              | NA                                                                                                                | 10.1002/pst.1934                  | Pharmaceutical statistics                                         | Milestone prediction for time-to-event endpoint monitoring in clinical trials                                                                                                                            |
| 2019 | MetaDTA                                      | https://crsu.shinyapps.io/dta_ma/                                                          | NA                                                                                                                | 10.1186/s12874-019-0724-x         | BMC Medical Research Methodology                                  | Development of an interactive web-based tool to conduct and interrogate meta-analysis of diagnostic test accuracy studies: MetaDTA                                                                       |
| 2019 | MetaInsight                                  | https://crsu.shinyapps.io/metainsightc                                                     | NA                                                                                                                | 10.1002/jrsm.1373                 | Research Synthesis Methods                                        | MetaInsight: An interactive web-based tool for analyzing, interrogating, and visualizing network meta-analyses using R-shiny and netmeta                                                                 |
| 2019 | MIENTURNET                                   | http://userver.bio.uniroma1.it/apps/mienturnet/                                            | NA                                                                                                                | 10.1186/s12859-019-3105-x         | BMC Bioinformatics                                                | MIENTURNET: an interactive web tool for microRNA-target enrichment and network-based analysis                                                                                                            |
| 2019 | miRFA                                        | https://emmbor.shinyapps.io/mirfa/                                                         | https://github.com/emmbor/miRFA                                                                                   | 10.1186/s12859-019-2974-3         | BMC Bioinformatics                                                | miRFA: an automated pipeline for microRNA functional analysis with correlation support from TCGA and TCPA expression data in pancreatic cancer                                                           |
| 2019 | MLVAtype                                     | https://ucl-irec-ctma.shinyapps.io/NGS-MLVA-TYPING/                                        | NA                                                                                                                | 10.1371/journal.pone.0225848      | Plos One                                                          | Backward compatibility of whole genome sequencing data with MLVA typing using a new MLVAtype shiny application for Vibrio cholerae                                                                       |
| 2019 | MultiSCED                                    | http://www.single-case.com/MultiSCED                                                       | NA                                                                                                                | 10.3758/s13428-019-01216-2        | Behavior Research Methods                                         | MultiSCED: A tool for (meta-)analyzing single-case experimental data with multilevel modeling                                                                                                            |
| 2019 | NG-Tax 2.0                                   | NA                                                                                         | http://wurssb.gitlab.io/ngtax/commandLine.html                                                                    | 10.3389/fgene.2019.01366          | Frontiers in Genetics                                             | NG-Tax 2.0: A Semantic Framework for High-Throughput Amplicon Analysis                                                                                                                                   |
| 2019 | NOJAH                                        | http://bbisr.shinyapps.winship.emory.edu/NOJAH/                                            | https://github.com/bbisr-shinyapps/NOJAH/                                                                         | 10.1371/journal.pone.0204542      | Plos One                                                          | NOJAH: NOt Just Another Heatmap for genome-wide cluster analysis                                                                                                                                         |
| 2019 | NOMePlot                                     | http://www.landeiralab.ugr.es/software                                                     | https://github.com/frequena/nomeplot                                                                              | 10.1038/s41598-019-44597-2        | Scientific Reports                                                | NOMePlot: analysis of DNA methylation and nucleosome occupancy at the single molecule                                                                                                                    |
| 2019 | One-Sample Power Calculator                  | https://mathr.math.appstate.edu/shiny/Power2/                                              | NA                                                                                                                | 10.1111/test.12186                | Teaching Statistics                                               | Using a Shiny app to teach the concept of power                                                                                                                                                          |
| 2019 | PatientExploreR                              | http://patientexplorer.ucsf.edu/                                                           | https://github.com/BenGlicksberg/PatientExploreR                                                                  | 10.1093/bioinformatics/btz409     | Bioinformatics                                                    | PatientExploreR: an extensible application for dynamic visualization of patient clinical history from Electronic Health Records in the OMOP Common Data Model Title                                      |
| 2019 | Pavian                                       | NA                                                                                         | http://github.com/fbreitwieser/pavian                                                                             | 10.1093/bioinformatics/btz715     | Bioinformatics                                                    | Pavian: Interactive analysis of metagenomics data for microbiome studies and pathogen identification                                                                                                     |
| 2019 | pcaExplorer                                  | http://bioconductor.org/packages/pcaExplorer/                                              | https://github.com/federicomarini/pcaExplorer/                                                                    | 10.1186/s12859-019-2879-1         | BMC Bioinformatics                                                | pcaExplorer: an R/Bioconductor package for interacting with RNA-seq principal components                                                                                                                 |
| 2019 | Pheno-seq                                    | NA                                                                                         | https://github.com/eilslabs/pheno-seq                                                                             | 10.1038/s41598-019-48771-4        | Scientific Reports                                                | Pheno-seq - linking visual features and gene expression in 3D cell culture systems                                                                                                                       |
| 2019 | PKconverter                                  | https://ek-lee.shinyapps.io/PKconverter/                                                   | https://cran.r-project.org/web/packages/PKconverter/index.html                                                    | 10.12793/tcp.2019.27.2.73         | Translational and clinical pharmacology                           | PKconverter: R package to convert the pharmacokinetic parameters                                                                                                                                         |
| 2019 | PlateDesigner                                | http://platedesigner.net/                                                                  | NA                                                                                                                | 10.1093/bioinformatics/bty853     | Bioinformatics                                                    | PlateDesigner: a web-based application for the design of microplate experiments                                                                                                                          |
| 2019 | PlotsOfData                                  | https://huygens.science.uva.nl/PlotsOfData/                                                | https://github.com/JoachimGoedhart/PlotsOfData                                                                    | 10.1371/journal.pbio.3000202      | Plos Biology                                                      | PlotsOfData-A web app for visualizing data together with their summaries                                                                                                                                 |
| 2019 | POLICI                                       | https://polici.shinyapps.io/yellow_fever_africa/                                           | NA                                                                                                                | 10.1016/j.vaccine.2019.01.074     | Vaccine                                                           | POLICI: A web application for visualising and extracting yellow fever vaccination coverage in Africa                                                                                                     |
| 2019 | PseudoFuN                                    | https://integrativeomics.shinyapps.io/pseudofun_app/                                       | https://github.com/OSC/pseudofun                                                                                  | 10.1093/gigascience/giz046        | GigaScience                                                       | PseudoFuN: Deriving functional potentials of pseudogenes from integrative relationships with genes and microRNAs across 32 cancers                                                                       |
| 2019 | pwrEWAS                                      | https://biostats-shinyr.kumc.edu/pwrEWAS/                                                  | https://github.com/stefangraw/pwrEWAS                                                                             | 10.1186/s12859-019-2804-7         | BMC Bioinformatics                                                | pwrEWAS: a user-friendly tool for comprehensive power estimation for epigenome wide association studies (EWAS)                                                                                           |
| 2019 | PWST                                         | https://ulbbf.shinyapps.io/pwst/                                                           | NA                                                                                                                | 10.35248/0974-276X.19.12.500      | Journal of proteomics & bioinformatics                            | Interactive Web Tool for Standardizing Proteomics Workflow for Liquid Chromatography-Mass Spectrometry Data                                                                                              |
| 2019 | QGprofiler                                   | https://qgprofiler.openanalytics.eu/app/QGprofiler                                         | NA                                                                                                                | 10.1186/s12859-019-2975-2         | BMC Bioinformatics                                                | Analyzing magnetic bead QuantiGene® Plex 2.0 gene expression data in high throughput mode using Qgprofiler                                                                                               |
| 2019 | RadaR                                        | https://ceefluz.shinyapps.io/radar/                                                        | https://github.com/ceefluz/radar                                                                                  | 10.2196/12843                     | Journal of Medical Internet Research                              | Rapid Analysis of Diagnostic and Antimicrobial Patterns in R (RadaR): Interactive Open-Source Software App for Infection Management and Antimicrobial Stewardship                                        |
| 2019 | RadialPheno                                  | NA                                                                                         | https://github.com/gcmariano/radialPheno                                                                          | 10.1002/aps3.1253                 | Applications in Plant Sciences                                    | RadialPheno: A tool for near-surface phenology analysis through radial layouts                                                                                                                           |
| 2019 | Rchimerism                                   | NA                                                                                         | https://github.com/BioHPC/Rchimerism                                                                              | 10.1016/j.jmoldx.2019.08.005      | The Journal of molecular diagnostics                              | Rchimerism: An R Package for Automated Chimerism Data Analysis                                                                                                                                           |
| 2019 | Regeneration Rosetta                         | https://gsajournals.figshare.com/articles/Supplemental_Material_for_Rau_et_al_2019/8856227 | https://github.com/andreamrau/rosetta                                                                             | 10.1534/g3.119.400729             | G3 (Bethesda, Md.)                                                | Regeneration Rosetta: An Interactive Web Application To Explore Regeneration-Associated Gene Expression and Chromatin Accessibility                                                                      |
| 2019 | Resumeq                                      | https://sites.anses.fr/fr/minisite/resumeq                                                 | NA                                                                                                                | 10.3389/fvets.2019.00135          | Frontiers in Veterinary Science                                   | Resumeq: A Novel Way of Monitoring Equine Diseases Through the Centralization of Necropsy Data                                                                                                           |
| 2019 | RhythmicAlly                                 | NA                                                                                         | https://github.com/abhilashlakshman/RhythmicAlly                                                                  | 10.1177/0748730419862474          | Journal of Biological Rhythms                                     | RhythmicAlly: Your R and Shiny–Based Open-Source Ally for the Analysis of Biological Rhythms                                                                                                             |
| 2019 | RiboStreamR                                  | NA                                                                                         | https://github.com/pjperki2/riboStreamR                                                                           | 10.1186/s12864-019-5700-7         | BMC Genomics                                                      | RiboStreamR: a web application for quality control, analysis, and visualization of Ribo-seq data                                                                                                         |
| 2019 | RITAN                                        | NA                                                                                         | https://github.com/MTZimmer/RITAN                                                                                 | 10.7717/peerj.6994                | PeerJ                                                             | RITAN: rapid integration of term annotation and network resources                                                                                                                                        |
| 2019 | RnaSeqSampleSize                             | http://cqs.mc.vanderbilt.edu/shiny/RnaSeqSampleSize/                                       | NA                                                                                                                | 10.1186/s12859-018-2191-5         | BMC Bioinformatics                                                | RnaSeqSampleSize: real data based sample size estimation for RNA sequencing                                                                                                                              |
| 2019 | SAFA-HsG                                     | https://SAFA-HsG.shinyapps.io/home/                                                        | NA                                                                                                                | 10.1007/s12539-019-00340-z        | Interdisciplinary Sciences, Computational Life Sciences           | Development of an Interactive Web Application Shiny App for Frequency Analysis on Homo sapiens Genome (SAFA-HsG)                                                                                         |
| 2019 | SEMA                                         | https://sema.research.cchmc.org                                                            | https://github.com/msolmazm/sema                                                                                  | 10.1093/bioinformatics/btz303     | Bioinformatics                                                    | Graphical data mining of cancer mechanisms with SEMA                                                                                                                                                     |
| 2019 | shinyapriori3                                | https://gkarahanadali.shinyapps.io/shinyapriori3/                                          | NA                                                                                                                | 10.17671/gazibtd.448245           | Journal of Information Technologies                               | A Dynamic Application of Market Basket Analysis with R and Shiny in The Electric Materials Sector                                                                                                        |
| 2019 | shinyBN                                      | https://jiajin.shinyapps.io/shinyBN/                                                       | https://github.com/JiajinChen/shinyBN                                                                             | 10.1186/s12859-019-3309-0         | BMC Bioinformatics                                                | shinyBN: an online application for interactive Bayesian network inference and visualization                                                                                                              |
| 2019 | shiny-pred                                   | https://gmu-binf.shinyapps.io/shiny-pred                                                   | https://github.com/mauricioob/shiny-pred                                                                          | 10.12688/f1000research.17669.1    | F1000Research                                                     | shiny-pred: a server for the prediction of protein disordered regions                                                                                                                                    |
| 2019 | Shiny-Seq                                    | https://schultzelab.shinyapps.io/Shiny-Seq/                                                | https://github.com/schultzelab/Shiny-Seq                                                                          | 10.1186/s13104-019-4471-1         | BMC Research Notes                                                | Shiny-Seq: advanced guided transcriptome analysis                                                                                                                                                        |
| 2019 | shinySIGHT                                   | NA                                                                                         | https://github.com/nguyens7/tidyNano                                                                              | 10.1371/journal.pone.0218270      | Plos One                                                          | Quantifying murine placental extracellular vesicles across gestation and in preterm birth data with tidyNano: A computational framework for analyzing and visualizing nanoparticle data in R             |
| 2019 | Simple ClinVar                               | http://simple-clinvar.broadinstitute.org/                                                  | http://github.com/dlal-group/Simple-ClinVar                                                                       | 10.1093/nar/gkz411                | Nucleic Acids Research                                            | Simple ClinVar: an interactive web server to explore and retrieve gene and disease variants aggregated in ClinVar database                                                                               |
| 2019 | Simple prediction model                      | https://statacademy.shinyapps.io/App_inaguma_20190717/                                     | NA                                                                                                                | 10.1371/journal.pone.0221352      | Plos One                                                          | Prediction model for cardiovascular events or all-cause mortality in incident dialysis patients                                                                                                          |
| 2019 | SkeletalVis                                  | http://phenome.manchester.ac.uk                                                            | www.github.com/soulj/SkeletalVis-Shiny                                                                            | 10.1093/bioinformatics/bty947     | Bioinformatics                                                    | SkeletalVis: an exploration and meta-analysis data portal of cross-species skeletal transcriptomics data                                                                                                 |
| 2019 | SMoLR                                        | NA                                                                                         | https://github.com/ErasmusOIC/SMoLR                                                                               | 10.1186/s12859-018-2578-3         | BMC Bioinformatics                                                | SMoLR: visualization and analysis of single-molecule localization microscopy data in R                                                                                                                   |
| 2019 | SpNetPrep                                    | https://albriz.shinyapps.io/spnetprep/                                                     | https://github.com/albrizre/SpNetPrep                                                                             | 10.3897/rio.5.e33521              | Research Ideas and Outcomes                                       | SpNetPrep: An R package using Shiny to facilitate spatial statistics on road networks                                                                                                                    |
| 2019 | SV-Pop                                       | NA                                                                                         | https://github.com/mattravenhall/SV-Pop                                                                           | 10.1186/s12859-019-2718-4         | BMC Bioinformatics                                                | SV-Pop: population-based structural variant analysis and visualization                                                                                                                                   |
| 2019 | TCC-GUI                                      | https://infinityloop.shinyapps.io/TCC-GUI/                                                 | https://github.com/swsoyee/TCC-GUI                                                                                | 10.1186/s13104-019-4179-2         | BMC Research Notes                                                | TCC-GUI: a Shiny-based application for differential expression analysis of RNA-Seq count data                                                                                                            |
| 2019 | TCGA                                         | http://ls-shiny-prod.uwm.edu/edge_in_tcga/                                                 | https://github.com/andreamrau/EDGE-in-                                                                            | 10.1093/bioinformatics/bty551     | Bioinformatics                                                    | Exploring drivers of gene expression in the Cancer Genome Atlas                                                                                                                                          |
| 2019 | Thermostats                                  | https://aswansyahputra.shinyapps.io/thermostats/                                           | https://github.com/dasaptaerwin/thermostats                                                                       | 10.30630/joiv.3.2-2.282           | JOIV                                                              | Thermostats: an Open Source Shiny App for Your Open Data Repository                                                                                                                                      |
| 2019 | timeseriesNEM                                | https://emt.bsse.ethz.ch                                                                   | https://github.com/cbg-ethz/timeseriesNEM                                                                         | 10.1093/bioinformatics/btz325     | Bioinformatics                                                    | Inferring signalling dynamics by integrating interventional with observational data                                                                                                                      |
| 2019 | Treehouse                                    | NA                                                                                         | https://github.com/JLSteenwyk/treehouse                                                                           | 10.1186/s13104-019-4577-5         | BMC Research Notes                                                | Treehouse: a user-friendly application to obtain subtrees from large phylogenies                                                                                                                         |
| 2019 | Ularcirc                                     | NA                                                                                         | https://github.com/VCCRI/Ularcirc                                                                                 | 10.1093/nar/gkz718                | Nucleic Acids Research                                            | Ularcirc: visualization and enhanced analysis of circular RNAs via back and canonical forward splicing                                                                                                   |
| 2019 | UltraMassExplorer                            | NA                                                                                         | NA                                                                                                                | 10.1002/rcm.8315                  | Rapid Communications in Mass Spectrometry : RCM                   | UltraMassExplorer: a browser-based application for the evaluation of high-resolution mass spectrometric data                                                                                             |
| 2019 | U-PASS                                       | https://power.stat.lsa.umich.edu/                                                          | https://github.com/Pill-GZ/U-PASS                                                                                 | 10.1093/bioinformatics/btz637     | Bioinformatics                                                    | U-PASS: unified power analysis and forensics for qualitative traits in genetic association studies                                                                                                       |
| 2019 | vancomycin TDM                               | http://pipet.shinyapps.io/vancomycin                                                       | https://github.com/pipetcpt/tdm-vancomycin                                                                        | 10.3390/pharmaceutics11050224     | Pharmaceutics                                                     | Application of Pharmacometrics in Pharmacotherapy: Open-Source Software for Vancomycin Therapeutic Drug Management                                                                                       |
| 2019 | VarSelLCM                                    | NA                                                                                         | https://cran.r-project.org/web/packages/VarSelLCM/index.html                                                      | 10.1093/bioinformatics/bty786     | Bioinformatics                                                    | VarSelLCM: an R/C++ package for variable selection in model-based clustering of mixed-data with missing values                                                                                           |
| 2019 | VisioProt-MS                                 | https://masstools.ipbs.fr/mstools/visioprot-ms/                                            | https://github.com/mlocardpaulet/VisioProt-MS                                                                     | 10.1093/bioinformatics/bty680     | Bioinformatics                                                    | VisioProt-MS: interactive 2D maps from intact protein mass spectrometry                                                                                                                                  |
| 2019 | VO₂FITTING                                   | https://shiny.cespu.pt/vo2_news/                                                           | NA                                                                                                                | 10.3390/sports7020031             | Sports                                                            | VO₂FITTING: A Free and Open-Source Software for Modelling Oxygen Uptake Kinetics in Swimming and other Exercise Modalities                                                                               |
| 2020 | 3D RNA-seq                                   | https://3drnaseq.hutton.ac.uk                                                              | https://github.com/wyguo/ThreeDRNAseq                                                                             | 10.1080/15476286.2020.1858253     | RNA biology                                                       | 3D RNA-seq: a powerful and flexible tool for rapid and accurate differential expression and alternative splicing analysis of RNA-seq data for biologists                                                 |
| 2020 | 8 Cell Line Protein Abundance Profile Viewer | https://xliu2020.shinyapps.io/8cells/                                                      | http://wren.hms.harvard.edu/8cells/                                                                               | 10.1002/pmic.202000218            | Proteomics                                                        | Isobaric Tag-Based Protein Profiling across Eight Human Cell Lines Using High-Field Asymmetric Ion Mobility Spectrometry and Real-Time Database Searching                                                |
| 2020 | ABCMETAapp                                   | NA                                                                                         | https://arxiv.org/ftp/arxiv/papers/2004/2004.02065.pdf                                                            | arXiv:2004.02065                  | arXiv                                                             | ABCMETAapp: R Shiny Application for Simulation-based Estimation of Mean and Standard Deviation for Meta-analysis via Approximate Bayesian Computation (ABC)                                              |
| 2020 | AGECalibratoR                                | https://drozdovapb.shinyapps.io/AGECalibratoR_2/                                           | https://github.com/drozdovapb/AGECalibratoR                                                                       | 10.1080/19336896.2020.1751574     | Prion                                                             | Estimation of amyloid aggregate sizes with semi-denaturing detergent agarosegel electrophoresis and its limitations                                                                                      |
| 2020 | aPCoA                                        | https://biostatistics.mdanderson.org/shinyapps/aPCoA/                                      | https://github.com/YushuShi/aPCoA                                                                                 | 10.1093/bioinformatics/btaa276    | Bioinformatics                                                    | aPCoA: covariate adjusted principal coordinates analysis                                                                                                                                                 |
| 2020 | ARPIR                                        | NA                                                                                         | https://github.com/giuliospinozzi/arpir                                                                           | 10.1186/s12859-020-03846-2        | BMC Bioinformatics                                                | ARPIR: automatic RNA-Seq pipelines with interactive repor                                                                                                                                                |
| 2020 | Ascall                                       | http://shtest.evrogen.net/AScal                                                            | https://github.com/kablag/AScall                                                                                  | 10.3389/fbioe.2020.00353          | Frontiers in bioengineering and biotechnology                     | AScall - Automatic Allele-Specific qPCR Analysis                                                                                                                                                         |
| 2020 | BedSect                                      | https://imgsb.org/bedsect/                                                                 | https://github.com/sraghav-lab/Bedsect                                                                            | 10.3389/fgene.2020.00003          | Frontiers in Genetics                                             | BedSect: An Integrated Web Server Application to Perform Intersection, Visualization, and Functional Annotation of Genomic Regions From Multiple Datasets                                                |
| 2020 | BEST                                         | https://www.brenda-enzymes.org/                                                            | https://github.com/StarBrand/BrendaSOAP                                                                           | 10.1093/bioinformatics/btaa848    | Bioinformatics                                                    | BEST: a Shiny/R web-based application to easily retrieve cross-related enzyme functional parameters and information from BRENDA                                                                          |
| 2020 | BGCViz                                       | https://ostash-group.shinyapps.io/BGCViz/                                                  | https://github.com/pavlohrab/BGCViz                                                                               | 10.13140/RG.2.2.23431.01444       | 4th Latin American Student Council Symposium                      | BGCViz: a shiny app for different BGC annotations visualization                                                                                                                                          |
| 2020 | bigPint                                      | https://bioconductor.org/packages/devel/bioc/html/bigPint.html                             | https://lindsayrutter.github.io/bigPint                                                                           | 10.1371/journal.pcbi.1007912      | PLoS Computational Biology                                        | bigPint: A Bioconductor visualization package that makes big data pint-sized                                                                                                                             |
| 2020 | BMDx                                         | NA                                                                                         | https://github.com/Greco-Lab/BMDx/                                                                                | 10.1093/bioinformatics/btaa030    | Bioinformatics                                                    | BMDx: a graphical Shiny application to perform Benchmark Dose analysis for transcriptomics data                                                                                                          |
| 2020 | BnaSNPDB                                     | https://bnapus-zju.com/bnasnpdb/                                                           | https://github.com/YTLogos/BnaSNPDB                                                                               | 10.1016/j.csbj.2020.09.031        | Computational and Structural Biotechnology Journal                | BnaSNPDB: An interactive web portal for the efficient retrieval and analysis of SNPs among 1,007 rapeseed accessions                                                                                     |
| 2020 | BREC                                         | NA                                                                                         | https://github.com/ymansour21/BREC                                                                                | 10.1101/2020.06.29.178095         | bioRxiv                                                           | BREC: An R package/Shiny app for automatically identifying heterochromatin boundaries and estimating local recombination rates along chromosomes                                                         |
| 2020 | BSAseq                                       | https://github.com/dblhlx/PyBSASeq                                                         | https://www.sciapps.org/page/bsa                                                                                  | 10.1093/bioinformatics/btaa709    | Bioinformatics                                                    | BSAseq: an interactive and integrated web-based workflow for identification of causal mutations in bulked F2 populations                                                                                 |
| 2020 | BUVARD online                                | http://www.irstea.fr/les-zones-tampons                                                     | NA                                                                                                                | hal-02605070                      | 5th international EcoSummit                                       | BUVARD online: a webtool using the R language (Shiny) to help end-users sizing their vegetative filter strips                                                                                            |
| 2020 | CELLector                                    | https://ot-cellector.shinyapps.io/CELLector_App/                                           | https://github.com/francescojm/CELLector and https://github.com/francescojm/CELLector_App                         | 10.1016/j.cels.2020.04.007        | Cell Systems                                                      | CELLector: Genomics-Guided Selection of Cancer In Vitro Models                                                                                                                                           |
| 2020 | Cerebro                                      | NA                                                                                         | https://github.com/romanhaa/Cerebro                                                                               | 10.1093/bioinformatics/btz877     | Bioinformatics                                                    | Cerebro: interactive visualization of scRNA-seq data                                                                                                                                                     |
| 2020 | Chloroplot                                   | https://irscope.shinyapps.io/chloroplot/                                                   | NA                                                                                                                | 10.3389/fgene.2020.576124         | Frontiers in Genetics                                             | Chloroplot: An Online Program for the Versatile Plotting of Organelle Genomes                                                                                                                            |
| 2020 | ChromSCape                                   | https://vallotlab.shinyapps.io/ChromSCape/                                                 | https://github.com/vallotlab/ChromSCape                                                                           | 10.1038/s41467-020-19542-x        | Nature Communications                                             | Interactive analysis of single-cell epigenomic landscapes with ChromSCape                                                                                                                                |
| 2020 | CIPR                                         | https://aekiz.shinyapps.io/CIPR/                                                           | https://github.com/atakanekiz/CIPR-Shiny                                                                          | 10.1186/s12859-020-3538-2         | BMC Bioinformatics                                                | CIPR: a web-based R/shiny app and R package to annotate cell clusters in single cell RNA sequencing experiments                                                                                          |
| 2020 | CIRCADA                                      | https://osf.io/seyhp/                                                                      | https://github.com/CenterForOpenScience/osfr                                                                      | 10.1177/0748730419900866          | Journal of Biological Rhythms                                     | CIRCADA: Shiny Apps for Exploration of Experimental and Synthetic Circadian Time Series with an Educational Emphasis                                                                                     |
| 2020 | clinical-dashboards                          | NA                                                                                         | https://github.com/mgolafshar/clinical-dashboards                                                                 | 10.1007/s11136-020-02617-z        | Quality of life research                                          | Utilizing open-source platforms to build and deploy interactive patient-reported quality of life tracking tools for monitoring protocol adherence                                                        |
| 2020 | clust.perturb                                | https://rstacey.shinyapps.io/clust-perturb-tool/                                           | https://github.com/fosterlab/clust-perturb                                                                        | 10.1074/mcp.RA120.002275          | Molecular and cellular proteomics                                 | On the robustness of graph-based clustering to random network alterations                                                                                                                                |
| 2020 | Cluster-RCT-Sample-Size-Calculator           | https://clusterrcts.shinyapp.io/rshinyapp/                                                 | https://github.com/karlahemming/Cluster-RCT-Sample-Size-Calculator                                                | 10.1093/ije/dyz237                | International Journal of Epidemiology                             | A tutorial on sample size calculation for multiple-period cluster randomized parallel, cross-over and stepped-wedge trials using the Shiny CRT Calculator                                                |
| 2020 | CONSENSUS                                    | https://aswansyahputra.shinyapps.io/thermostats/                                           | https://github.com/dasaptaerwin/thermostats                                                                       | 10.21203/rs.3.rs-69396/v2         | Research Square                                                   | CONSENSUS: A Shiny application of Dementia evaluation and reporting for the KU ADC Longitudinal Clinical Cohort Database                                                                                 |
| 2020 | COVID-19 - España                            | https://covid19.citic.udc.es/                                                              | NA                                                                                                                | 10.3390/proceedings2020054023     | Proceedings                                                       | Shiny Dashboard for Monitoring the COVID-19 Pandemic in Spain                                                                                                                                            |
| 2020 | COVID-19 Watcher                             | https://covid19watcher.research.cchmc.org/                                                 | https://github.com/wisselbd/COVID-Tracker                                                                         | 10.1093/jamia/ocaa071             | Journal of the American Medical Informatics Association           | An interactive online dashboard for tracking COVID-19 in U.S. counties, cities, and states in real time                                                                                                  |
| 2020 | Covid19Dashboard                             | https://mahdisalehi.shinyapps.io/Covid19Dashboard/                                         | https://github.com/Mahdi-Salehi-PhD/COVID-19-dashboard                                                            | 10.3389/fpubh.2020.623624         | Front Public Health                                               | A Synergetic R-Shiny Portal for Modeling and Tracking of COVID-19 Data                                                                                                                                   |
| 2020 | COVID19-Tracker                              | https://ubidi.shinyapps.io/covid19/                                                        | NA                                                                                                                | 10.1101/2020.04.01.20049684       | medRxiv                                                           | COVID19-Tracker: A shiny app to perform comprehensive data visualisation for SARS-CoV-2 epidemic in Spain                                                                                                |
| 2020 | COVID19-world                                | https://ubidi.shinyapps.io/covid19world                                                    | https://github.com/ubidi/covid19world                                                                             | 10.1186/s12874-020-01121-9        | BMC Medical Research Methodology                                  | COVID19-world: a shiny application to perform comprehensive country-specific data visualization for SARS-CoV-2 epidemic                                                                                  |
| 2020 | CytofRUV                                     | NA                                                                                         | https://github.com/mtrussart/CytofRUV/                                                                            | 10.1101/2020.05.09.085621         | Elife                                                             | Removing unwanted variation with CytofRUV to integrate multiple CyTOF datasets                                                                                                                           |
| 2020 | Cytomapper                                   | NA                                                                                         | https://github.com/BodenmillerGroup/cytomapper_publication                                                        | 10.1093/bioinformatics/btaa1061   | Bioinformatics                                                    | Cytomapper: an R/bioconductor package for visualisation of highly multiplexed imaging data                                                                                                               |
| 2020 | DeCovid                                      | NA                                                                                         | https://github.com/ConesaLab/DeCovid                                                                              | 10.3390/genes11121447             | Genes                                                             | Transcriptional Differences for COVID-19 Disease Map Genes between Males and Females Indicate a Different Basal Immunophenotype Relevant to the Disease                                                  |
| 2020 | DNA_Activity                                 | https://lydie-samie.shinyapps.io/DNA_Activity/                                             | NA                                                                                                                | 10.1016/j.fsigen.2020.102334      | Forensic science international. Genetics                          | The use of Bayesian Networks and simulation methods to identify the variables impacting the value of evidence assessed under activity level propositions in stabbing cases                               |
| 2020 | DolphinNext                                  | https://dolphinnext.umassmed.edu/                                                          | https://github.com/UMMS-Biocore/                                                                                  | 10.1186/s12864-020-6714-x         | BMC Genomics                                                      | DolphinNext: a distributed data processing platform for high throughput genomics                                                                                                                         |
| 2020 | DosePredict                                  | https://mokour.shinyapps.io/DosePredict/                                                   | https://github.com/malekokour1/DosePredict                                                                        | 10.1002/jcph.1649                 | Pharmacokinetics                                                  | DosePredict: A Shiny Application for Generalized Pharmacokinetics‐Based Dose Predictions                                                                                                                 |
| 2020 | DscoreApp                                    | https://fisppa.psy.unipd.it/DscoreApp/                                                     | https://github.com/OttaviaE/DscoreApp                                                                             | 10.3389/fpsyg.2019.02938          | Frontiers in psychology                                           | DscoreApp: A Shiny Web Application for the Computation of the Implicit Association Test D-Score                                                                                                          |
| 2020 | ECCDIA                                       | http://webapps.3steps.cn/ECCDIA/                                                           | https://github.com/clinico-omics/ECCDIA                                                                           | 10.1186/s12885-020-07479-9        | BMC Cancer                                                        | ECCDIA: an interactive web tool for the comprehensive analysis of clinical and survival data of esophageal cancer patients                                                                               |
| 2020 | eCovid19 - CIIBI                             | http://covid19.ciibi.mx/                                                                   | NA                                                                                                                | 10.3390/ijerph17145173            | International journal of environmental research and public health | Demographic and Comorbidities Data Description of Population in Mexico with SARS-CoV-2 Infected Patients(COVID19): An Online Tool Analysis                                                               |
| 2020 | EDSS                                         | NA                                                                                         | https://github.com/cocomice/EDSS_dep_framework                                                                    | 10.1016/j.envsoft.2020.104797     | Environmental Modelling & Software                                | Towards fast prototyping of cloud-based environmental decision support systems for environmental scientists using R Shiny and Docker                                                                     |
| 2020 | EHRtemporalVariability                       | http://ehrtemporalvariability.upv.es/                                                      | https://github.com/hms-dbmi/EHRtemporalVariability/Reproducible                                                   | 10.1093/gigascience/giaa079       | GigaScience                                                       | EHRtemporalVariability: delineating temporal data-set shifts in electronic health records                                                                                                                |
| 2020 | endo_eqtl_rna                                | http://reproductivegenomics.com.au/shiny/endo_eqtl_rna/                                    | NA                                                                                                                | 10.1093/humrep/dez279             | Human Reproduction                                                | Tissue specific regulation of transcription in endometrium and association with disease                                                                                                                  |
| 2020 | EpiDISH                                      | https://www.biosino.org/EpiDISH/                                                           | NA                                                                                                                | 10.1093/bioinformatics/btz833     | Bioinformatics                                                    | EpiDISH web server: Epigenetic Dissection of Intra-Sample-Heterogeneity with online GUI                                                                                                                  |
| 2020 | equivalence-of-variances                     | https://optimal-sample-size.shinyapps.io/equivalence-of-variances/                         | NA                                                                                                                | 10.1111/bmsp.12172                | British Journal of Mathematical and Statistical Psychology        | Testing two variances for superiority/non‐inferiority and equivalence: Using the exhaustion algorithm for sample size allocation with cost                                                               |
| 2020 | ExploreModelMatrix                           | http://shiny.imbei.uni-mainz.de:3838/ExploreModelMatrix                                    | https://github.com/csoneson/ExploreModelMatrix                                                                    | 10.12688/f1000research.24187.2    | F1000Research                                                     | ExploreModelMatrix: Interactive exploration for improved understanding of design matrices and linear models in R                                                                                         |
| 2020 | FactorViz                                    | NA                                                                                         | https://github.com/CompEpigen/FactorViz                                                                           | 10.1038/s41596-020-0369-6         | Nature protocols                                                  | Reference-free deconvolution, visualization and interpretation of complex DNA methylation data using DecompPipeline, MeDeCom and FactorViz                                                               |
| 2020 | FaDA                                         | https://shiny-bird.univ-nantes.fr/app/Fada                                                 | https://github.com/danger-r/FaDAapp                                                                               | 10.1101/2020.04.23.055673         | bioRxiv                                                           | FaDA: A Shiny web application to accelerate common lab data analyses                                                                                                                                     |
| 2020 | Forensic Classification                      | https://dianagiurghita.shinyapps.io/ForensicClassification/                                | https://github.com/DianaGiurghita/Forensic-Classification                                                         | 10.3389/fchem.2020.00738          | Frontiers in Chemistry                                            | Evaluation of Forensic Data Using Logistic Regression-Based Classification Methods and an R Shiny Implementation                                                                                         |
| 2020 | GenePiper                                    | https://github.com/raytonghk/GenePiper                                                     | NA                                                                                                                | 10.1128/mra.01195-19              | Microbiology Resource Announcements                               | GenePiper, a Graphical User Interface Tool for Microbiome Sequence Data Mining                                                                                                                           |
| 2020 | GenomeQC                                     | https://genomeqc.maizegdb.org/                                                             | https://github.com/HuffordLab/GenomeQC                                                                            | 10.1186/s12864-020-6568-2         | BMC Genomics                                                      | GenomeQC: a quality assessment tool for genome assemblies and gene structure annotations                                                                                                                 |
| 2020 | GMIEC                                        | NA                                                                                         | https://github.com/guidmt/GMIEC-shiny                                                                             | 10.1186/s12864-020-06996-y        | BMC Genomics                                                      | GMIEC: a shiny application for the identification of gene-targeted drugs for precision medicine                                                                                                          |
| 2020 | GOF Poisson                                  | http://manu2h.shinyapps.io/gof_poisson/                                                    | NA                                                                                                                | 10.1097/hp.0000000000001285       | Health Physics                                                    | Automated Dicentric Aberration Scoring for Triage Dose Assessment                                                                                                                                        |
| 2020 | gwaRs                                        | https://gwasviz.shinyapps.io/gwaRs/                                                        | https://github.com/LindoNkambule/gwaRs                                                                            | 10.1101/2020.04.17.044784         | bioRxiv                                                           | gwaRs: an R shiny web application for visualizing genome-wide association studies data                                                                                                                   |
| 2020 | HPAStainR                                    | https://32tim32.shinyapps.io/HPAStainR/                                                    | https://github.com/tnieuwe/HPAStainR                                                                              | 10.12688/f1000research.26771.1    | F1000Research                                                     | HPAStainR: a Bioconductor and Shiny app to query protein expression patterns in the Human Protein Atlas                                                                                                  |
| 2020 | HTrack                                       | NA                                                                                         | https://github.com/dmrodz/htrack                                                                                  | 10.1371/journal.pone.0244028      | Plos One                                                          | HTrack: A new tool to facilitate public health field visits and electronic data capture                                                                                                                  |
| 2020 | Human Microbiome Analysis                    | NA                                                                                         | https://github.com/microgenomics/HumanMicrobiomeAnalysis                                                          | 10.7717/peerj.9688                | PeerJ                                                             | Evaluation of computational methods for human microbiome analysis using simulated data                                                                                                                   |
| 2020 | ideal                                        | http://bioconductor.org/packages/ideal/                                                    | https://github.com/federicomarini/ideal/                                                                          | 10.1186/s12859-020-03819-5        | BMC Bioinformatics                                                | ideal: an R/Bioconductor package for interactive differential expression analysis                                                                                                                        |
| 2020 | idem                                         | NA                                                                                         | https://CRAN.R-project.org/package=idem                                                                           | 10.18637/jss.v093.i12             | Journal of statistical software                                   | idem: An R Package for Inferences in Clinical Trials with Death and Missingness                                                                                                                          |
| 2020 | iMutSig                                      | https://zhiyang.shinyapps.io/imutsig/                                                      | https://github.com/USCbiostats/iMutSig                                                                            | 10.12688/f1000research.24435.2    | F1000Research                                                     |  iMutSig: a web application to identify the most similar mutational signature using shiny                                                                                                                |
| 2020 | INSPEcT-GUI                                  | NA                                                                                         | http://bioconductor.org/packages/INSPEcT/                                                                         | 10.3389/fgene.2020.00759          | Frontiers in Genetics                                             | INSPEcT-GUI Reveals the Impact of the Kinetic Rates of RNA Synthesis, Processing, and Degradation, on Premature and Mature RNA Species                                                                   |
| 2020 | Interactive                                  | https://virusresp.xyz/virus/                                                               | https://gitlab.com/morksoldat/vigilancia-virus-respiratorios                                                      | 10.4067/S0716-10182020000100069   | Revista chilena de infectología                                  | Interactive, semi-automatized and open source computational model applied to respiratory viruses surveillance                                                                                            |
| 2020 | IR-Tex                                       | http://www.lstmed.ac.uk/projects/IR-Tex                                                    | http://github.com/LSTMScientificComputing/IR-Tex                                                                  | 10.3791/60721                     | Journal of Visualized Experiments                                 | IR-TEx: An Open Source Data Integration Tool for Big Data Transcriptomics Designed for the Malaria Vector Anopheles gambiae                                                                              |
| 2020 | iSEE                                         | https://shiny.mdc-berlin.de/MyoExplorer/                                                   | https://github.com/BIMSBbioinfo/MyoExplorer                                                                       | 10.1038/s41467-020-20064-9        | Nature Communications                                             | Single-nucleus transcriptomics reveals functional compartmentalization in syncytial skeletal muscle cells                                                                                                |
| 2020 | iSeqQC                                       | http://cancerwebpa.jefferson.edu/iSeqQC                                                    | https://github.com/gkumar09/iSeqQC                                                                                | 10.1186/s12859-020-3399-8         | BMC Bioinformatics                                                | iSeqQC: a tool for expression-based quality control in RNA sequencing                                                                                                                                    |
| 2020 | iu-ma                                        | http://www.iu-ma.org/                                                                      | https://github.com/iu-ma/bpmpmri                                                                                  | 10.1007/s10278-019-00312-1        | Journal of digital imaging                                        | Interactive, Up-to-date Meta-Analysis of MRI in the Management of Men with Suspected Prostate Cancer                                                                                                     |
| 2020 | KairosMS                                     | NA                                                                                         | NA                                                                                                                | 10.1021/acs.analchem.9b05113      | Analytical Chemistry                                              | KairosMS: A New Solution for the Processing of Hyphenated Ultrahigh Resolution Mass Spectrometry Data                                                                                                    |
| 2020 | Kaleidoscope                                 | https://kalganem.shinyapps.io/Kaleidoscope/                                                | NA                                                                                                                | 10.1101/2020.05.01.070805         | bioRxiv                                                           | Kaleidoscope: A New Bioinformatics Pipeline Web Application for In Silico Hypothesis Exploration of Omics Signatures                                                                                     |
| 2020 | LFDREmpiricalBayes                           | https://empiricalbayes.shinyapps.io/lfdrempiricalbayesapp                                  | https://cran.r-project.org/web/packages/LFDREmpiricalBayes                                                        | 10.1109/TCBB.2018.2865420         | IEEE/ACM Trans Comput Biol Bioinform                              | Incorporating Prior Knowledge about Genetic Variants into the Analysis of Genetic Association Data: An Empirical Bayes Approach                                                                          |
| 2020 | MDMAPR                                       | https://hannerlab.shinyapps.io/MDMAPR/                                                     | https://github.com/HannerLab/MDMAPR                                                                               | 10.7717/peerj.9974                | PeerJ                                                             | Molecular Detection Mapping and Analysis Platform for R (MDMAPR) facilitating the standardization, analysis, visualization, and sharing of qPCR data and metadata                                        |
| 2020 | Medullo Classifier                           | https://komalrathi.shinyapps.io/medulloclassifiershinyapp/                                 | https://github.com/d3b-center/medullo-classifier-package                                                          | 10.1371/journal.pcbi.1008263      | PLoS Computational Biology                                        | A transcriptome-based classifier to determine molecular subtypes in medulloblastoma                                                                                                                      |
| 2020 | MEPHAS                                       | https://alain003.phs.osaka-u.ac.jp/mephas/                                                 | https://mephas.github.io/                                                                                         | 10.1186/s12859-020-3494-x         | BMC Bioinformatics                                                | MEPHAS: an interactive graphical user interface for medical and pharmaceutical statistical analysis with R and Shiny                                                                                     |
| 2020 | Metabolite AutoPlotter                       | https://mpietzke.shinyapps.io/autoplotter.                                                 | NA                                                                                                                | 10.1186/s40170-020-00220-x        | Cancer & Metabolism                                               | Metabolite AutoPlotter - an application to process and visualise metabolite data in the web browser                                                                                                      |
| 2020 | Metabolite-Investigator                      | https://apps.health-atlas.de/metabolite-investigator/                                      | https://github.com/cfbeuchel/Metabolite-Investigator                                                              | 10.1093/bioinformatics/btaa967    | Bioinformatics                                                    | Metabolite-Investigator: An integrated userfriendly workflow for metabolomics multi-study analysis                                                                                                       |
| 2020 | MetaboShiny                                  | NA                                                                                         | https://github.com/joannawolthuis/MetaboShiny                                                                     | 10.1101/734236                    | bioRxiv                                                           | MetaboShiny – interactive processing, analysis and annotation of direct infusion metabolomics data                                                                                                       |
| 2020 | methylscaper                                 | http://www.methylscaper.com/                                                               | https://github.com/rhondabacher/methylscaper                                                                      | 10.1101/2020.11.13.382465         | bioRxiv                                                           | methylscaper: an R/Shiny app for joint visualization of DNA methylation and nucleosome occupancy in single-molecule and single-cell data                                                                 |
| 2020 | MicrobiomeExplorer                           | NA                                                                                         | https://github.com/zoecastillo/microbiomeExplorer                                                                 | 10.1093/bioinformatics/btaa838    | Bioinformatics                                                    | MicrobiomeExplorer: An R package for the analysis and visualization of microbial communities                                                                                                             |
| 2020 | mmaqshiny                                    | NA                                                                                         | https://github.com/meenakshi-kushwaha/mmaqshiny                                                                   | 10.21105/joss.02250               | The Journal of Open Source Software                               | mmaqshiny v1.0: R-Shiny package to explore Air-Quality Mobile-Monitoring data                                                                                                                            |
| 2020 | MMPdb and MitoPredictor                      | https://mmpdb.eeob.iastate.edu/                                                            | https://github.com/virajmuthye/mitopredictor                                                                      | 10.1016/j.mito.2020.01.001        | Mitochondrion                                                     | MMPdb and MitoPredictor: Tools for facilitating comparative analysis of animal mitochondrial proteomes                                                                                                   |
| 2020 | MoDEsT                                       | https://medstats-lancs.shinyapps.io/design/                                                | https://cran.r-project.org/web/packages/modest/vignettes/Design.html                                              | 10.1177/1740774519890146          | Clinical Trials                                                   | Designing and evaluating dose-escalation studies made easy: The MoDEsT web app                                                                                                                           |
| 2020 | MRT-SS Calculator                            | https://pengliao.shinyapps.io/mrt-calculator                                               | https://github.com/nseewald1/mrt-ss-calculator                                                                    | arXiv:1609.00695                  | arXiv                                                             | MRT-SS Calculator: An R Shiny Application for Sample Size Calculation in Micro-Randomized Trials                                                                                                         |
| 2020 | MS-CleanR                                    | http://prime.psc.riken.jp/compms/index.html                                                | https://github.com/eMetaboHUB/MS-CleanR                                                                           | 10.1021/acs.analchem.0c01594      | Analytical Chemistry                                              | MS-CleanR: A feature-filtering workflow for untargeted LC-MS based metabolomics                                                                                                                          |
| 2020 | multiarm                                     | https://mjgrayling.shinyapps.io/multiarm/                                                  | https://github.com/mjg211/multiarm                                                                                | 10.1186/s12885-020-6525-0         | BMC Cancer                                                        | A web application for the design of multi-arm clinical trials                                                                                                                                            |
| 2020 | MultiFlow                                    | NA                                                                                         | https://github.com/stamats/MultiFlow                                                                              | 10.1186/s12951-020-00688-1        | Journal of Nanobiotechnology                                      | Duplex Shiny app quantification of the sepsis biomarkers C-reactive protein and interleukin-6 in a fast quantum dot labeled lateral flow assay                                                           |
| 2020 | MutRank                                      | NA                                                                                         | https://github.com/eporetsky/MutRank                                                                              | 10.7717/peerj.10264               | PeerJ                                                             | MutRank: an R shiny web-application for exploratory targeted mutual rank-based coexpression analyses integrated with user-provided supporting information                                                |
| 2020 | MVC1_R                                       | https://atmunr.shinyapps.io/MVC1_R/                                                        | https://github.com/atmunr/MVC1_R                                                                                  | 10.1021/acs.jchemed.9b00850       | Journal of chemical education                                     | Developing and Implementing an R Shiny Application to Introduce Multivariate Calibration to Advanced Undergraduate Students                                                                              |
| 2020 | netGO                                        | NA                                                                                         | https://github.com/unistbig/netGO                                                                                 | 10.1093/bioinformatics/btaa077    | Bioinformatics                                                    | netGO: R-Shiny package for network-integrated pathway enrichment analysis                                                                                                                                |
| 2020 | omicsBioAnalytics                            | https://amritsingh.shinyapps.io/omicsBioAnalytics/                                         | https://github.com/singha53/omicsBioAnalytics                                                                     | 10.1101/2020.05.05.024323         | bioRxiv                                                           | Omics BioAnalytics: Reproducible Research using R Shiny and Alexa                                                                                                                                        |
| 2020 | ORGANIZATIONAL SCIENCE                       | https://orgscience.uncc.edu/about-us/resources                                             | NA                                                                                                                | 10.1037/apl0000815                | The Journal of applied psychology                                 | From alpha to omega and beyond! A look at the past, present, and (possible) future of psychometric soundness in the Journal of Applied Psychology                                                        |
| 2020 | PartiallyNestedMediationPower                | https://poweruprshiny.shinyapps.io/PartiallyNestedMediationPower/                          | NA                                                                                                                | 10.1080/10503307.2020.1717012     | Psychotherapy research                                            | Statistical power in partially nested designs probing multilevel mediation                                                                                                                               |
| 2020 | PAST                                         | NA                                                                                         | https://github.com/IGBB/PAST/releases                                                                             | 10.3791/61268                     | Journal of Visualized Experiments                                 | A Pathway Association Study Tool for GWAS Analyses of Metabolic Pathway Information                                                                                                                      |
| 2020 | PAST                                         | https://rdrr.io/bioc/PAST/                                                                 | NA                                                                                                                | 10.3390/plants9010058             | Plants (Basel, Switzerland)                                       | PAST: The Pathway Association Studies Tool to Infer Biological Meaning from GWAS Datasets                                                                                                                |
| 2020 | pepFunk                                      | https://shiny.imetalab.ca/pepFunk/                                                         | https://github.com/northomics/pepFunk                                                                             | 10.1093/bioinformatics/btaa289    | Bioinformatics                                                    | pepFunk: a tool for peptide-centric functional analysis of metaproteomic human gut microbiome studies                                                                                                    |
| 2020 | PeskAAS                                      | https://doi.org/10.7910/DVN/TVGIJJ                                                         | https://github.com/shaunpwilkinson/PeskAAS                                                                        | 10.1371/journal.pone.0234760      | Plos One                                                          | PeskAAS: A near-real-time, open-source monitoring and analytics system for small-scale fisheries                                                                                                         |
| 2020 | PhenoWin                                     | NA                                                                                         | https://github.com/EMRAgit/PhenoWin                                                                               | 10.1016/j.compag.2020.105534      | Computers and Electronics in Agriculture                          | PhenoWin – An R Shiny application for visualization and extraction of phenological windows in Germany                                                                                                    |
| 2020 | Pivot Science                                | http://research.pivot-dashboard.org:3838/                                                  | NA                                                                                                                | 10.1186/s12942-020-00220-6        | International journal of health geographics                       | Improving geographical accessibility modeling for operational use by local health actors                                                                                                                 |
| 2020 | power4SEM                                    | https://sjak.shinyapps.io/power4SEM/                                                       | https://osf.io/39gx8/                                                                                             | 10.3758/s13428-020-01479-0        | Behavior Research Methods                                         | Analytical power calculations for structural equation modeling: A tutorial and Shiny app                                                                                                                 |
| 2020 | powerEQTL                                    | https://bwhbioinfo.shinyapps.io/powerEQTL/                                                 | https://github.com/sterding/powerEQTL                                                                             | 10.1101/2020.12.15.422954         | bioRxiv                                                           | powerEQTL: An R package and shiny application for sample size and power calculation of bulk tissue and single-cell eQTL analysis                                                                         |
| 2020 | Preconditioning and damage transcriptome     | http://shiny.cecad.uni-koeln.de:3838/IRaP/                                                 | NA                                                                                                                | 10.1681/ASN.2019050534            | Journal of the American Society of Nephrology                     | The Integrated RNA Landscape of Renal Preconditioning against Ischemia-Reperfusion Injury                                                                                                                |
| 2020 | prevalence-tool                              | https://www.ambrygen.com/prevalence-tool                                                   | NA                                                                                                                | 10.1002/humu.24053                | Human Mutation                                                    | Mutation prevalence tables for hereditary cancer derived from multigene panel testing                                                                                                                    |
| 2020 | propensityscorematching                      | https://propensityscorematching.shinyapps.io/propensityscorematching/                      | NA                                                                                                                | 10.5455/annalsmedres.2020.01.047  | Annals of Medical Research                                        | An interactive web application for propensity score matching with R shiny; example of thrombophilia                                                                                                      |
| 2020 | qgshiny                                      | http://neyhartj.shinyapps.io/qgshiny                                                       | https://github.com/neyhartj/qgshiny                                                                               | 10.1002/nse2.20026                | Natural Sciences Education                                        | An active learning tool for quantitative genetics instruction using R and shiny                                                                                                                          |
| 2020 | rapidPop                                     | NA                                                                                         | https://github.com/mikeyEcology/rapidPop                                                                          | 10.1101/2020.03.30.017103         | bioRxiv                                                           | rapidPop: Rapid population assessments of wildlife using camera trap data in R Shiny Applications                                                                                                        |
| 2020 | ReMap                                        | http://remap.univ-amu.fr                                                                   | https://github.com/remap-cisreg                                                                                   | 10.1093/nar/gkz945                | Nucleic Acids Research                                            | ReMap 2020: a database of regulatory regions from an integrative analysis of Human and Arabidopsis DNA-binding sequencing experiment                                                                     |
| 2020 | robvis                                       | https://www.riskofbias.info/                                                               | https://github.com/mcguinlu/robvis                                                                                | 10.1002/jrsm.1411                 | Research Synthesis Methods                                        | Risk-of-bias VISualization (robvis): An R package and Shiny web app for visualizing risk‐of‐bias assessments                                                                                             |
| 2020 | SARS-CoV-2                                   | https://theguarani.com.br/                                                                 | https://github.com/adamtaiti/SARS-CoV-2                                                                           | 10.3389/fmed.2020.00247           | Frontiers in Medicine                                             | Growth Rate and Acceleration Analysis of the COVID-19 Pandemic Reveals the Effect of Public Health Measures in Real Time                                                                                 |
| 2020 | SCHNAPPs                                     | https://c3bi-pasteur-fr.github.io/UTechSCB-SCHNAPPs                                        | https://github.com/baj12/SCHNAPPsContributions                                                                    | 10.1101/2020.06.07.127274         | bioRxiv                                                           | SCHNAPPs - Single Cell sHiNy APPlication(s)                                                                                                                                                              |
| 2020 | SCTCwhatateam                                | NA                                                                                         | https://github.com/pvvhoang/SCTCwhatateam-ShinyApp                                                                | 10.1093/bib/bbaa181               | Briefings in bioinformatics                                       | The winning methods for predicting cellular position in the DREAM single-cell transcriptomics challenge                                                                                                  |
| 2020 | sepsismap                                    | https://sepsismap.shinyapps.io/index2/                                                     | NA                                                                                                                | 10.1007/s10916-019-1509-9         | Journal of medical systems                                        | Web-Based Dashboard for the Interactive Visualization and Analysis of National Risk-Standardized Mortality Rates of Sepsis in the US                                                                     |
| 2020 | Sequential t-SNE                             | https://chpupsom19.shinyapps.io/Survival_Analysis_tsne_umap_TCGA                           | https://github.com/RavulaPitt/Sequential-t-SNE/                                                                   | 10.1186/s12885-020-06756-x        | BMC Cancer                                                        | Sequential analysis of transcript expression patterns improves survival prediction in multiple cancers                                                                                                   |
| 2020 | ShinyButchR                                  | https://hdsu-bioquant.shinyapps.io/shinyButchR/                                            | https://github.com/wurst-theke/ButchR                                                                             | 10.1093/biomethods/bpaa022        | Biology Methods and Protocols                                     | ShinyButchR: Interactive NMF-based decomposition workflow of genome-scale datasets                                                                                                                       |
| 2020 | shinyCurves                                  | https://biosol.shinyapps.io/shinycurves/                                                   | https://github.com/biosol/shinyCurves                                                                             | 10.1101/2020.10.14.20212381       | medRxiv                                                           | shinyCurves, a shiny web application to analyse multisource qPCR amplification data: a COVID 19 case study                                                                                               |
| 2020 | Shiny‑DEG                                    | NA                                                                                         | https://githu b.com/sufan gwang -npu/shiny -DEG.Users                                                             | 10.1007/s12539-020-00383-7        | Interdisciplinary Sciences, Computational Life Sciences           | Shiny‑DEG: A Web Application to Analyze and Visualize Differentially Expressed Genes in RNA‑seq                                                                                                          |
| 2020 | shinyFuse                                    | http://shiny.imbei.uni-mainz.de:3838/shinyFuse/                                            | https://github.com/d3b-center/annoFuse                                                                            | 10.1186/s12859-020-03922-7        | BMC Bioinformatics                                                | annoFuse: an R Package to annotate, prioritize, and interactively explore putative oncogenic RNA fusions                                                                                                 |
| 2020 | shinyNgsreports                              | NA                                                                                         | https://github.com/UofABioinformaticsHub/shinyNgsreports                                                          | 10.1093/bioinformatics/btz937     | Bioinformatics                                                    | ngsReports: a Bioconductor package for managing FastQC reports and other NGS related log files                                                                                                           |
| 2020 | Shiny-SoSV                                   | https://hcpcg.shinyapps.io/shiny-sosv/                                                     | https://github.com/tgong1/Shiny-SoSV                                                                              | 10.1371/journal.pone.0238108      | Plos One                                                          | Shiny-SoSV: A web-based performance calculator for somatic structural variant detection                                                                                                                  |
| 2020 | smsPOMDP                                     | https://conservation-decisions.shinyapps.io/smsPOMDP/                                      | https://github.com/conservation-decisions/smsPOMDP                                                                | 10.1111/2041-210X.13501           | Methods in Ecology and Evolution                                  | A Shiny r app to solve the problem of when to stop managing or surveying species under imperfect detection                                                                                               |
| 2020 | SnpHub                                       | http://wheat.cau.edu.cn/Wheat_SnpHub_Portal/                                               | http://guoweilong.github.io/SnpHub/                                                                               | 10.1093/gigascience/giaa060       | GigaScience                                                       | SnpHub: an easy-to-set-up web server framework for exploring large-scale genomic variation data in the post-genomic era with applications in wheat                                                       |
| 2020 | SOCRATES                                     | http://www.socialcontactdata.org                                                           | NA                                                                                                                | 10.1186/s13104-020-05136-9        | BMC Research Notes                                                | SOCRATES: an online tool leveraging a social contact data sharing initiative to assess mitigation strategies for COVID-19                                                                                |
| 2020 | Spectacle                                    | http://oculargeneexpression.org/singlecell                                                 | https://github.com/drewvoigt10/cellcuratoR                                                                        | 10.1016/j.exer.2020.108204        | Experimental eye research                                         | Spectacle: An interactive resource for ocular single-cell RNA sequencing data analysis                                                                                                                   |
| 2020 | START-R                                      | http://localhost:3839/                                                                     | https://github.com/thomasdenecker/START-R                                                                         | 10.1093/nargab/lqaa045            | NAR genomics and bioinformatics                                   | Efficient, quick and easy-to-use DNA replication timing analysis with START-R suite                                                                                                                      |
| 2020 | StructuRly                                   | https://nicocriscuolo.shinyapps.io/StructuRly/                                             | https://github.com/nicocriscuolo/StructuRly                                                                       | 10.1371/journal.pone.0229330      | Plos One                                                          | StructuRly: A novel shiny app to produce comprehensive, detailed and interactive plots for population genetic analysis                                                                                   |
| 2020 | SUMMER                                       | http://summer.salk.edu/                                                                    | https://bitbucket.org/salkigc/summer/src/master/                                                                  | 10.1007/s11306-020-01750-7        | Metabolomics                                                      | SUMMER, a shiny utility for metabolomics and multiomics exploratory research                                                                                                                             |
| 2020 | SURF                                         | http://www.statlab.wisc.edu/shiny/surf/                                                    | https://github.com/keleslab/surf                                                                                  | 10.1186/s13059-020-02039-7        | Genome Biology                                                    | SURF: integrative analysis of a compendium of RNA-seq and CLIP-seq datasets highlights complex governing of alternative transcriptional regulation by RNA-binding proteins                               |
| 2020 | swCRTdesign                                  | https://swcrtdesign.shinyapps.io/stepped_wedge_power_calculation/                          | https://github.com/swCRTdesign/Stepped-wedge-power-calculation                                                    | 10.1016/j.cmpb.2020.105514        | Computer methods and programs in biomedicine                      | swCRTdesign: An RPackage for Stepped Wedge Trial Design and Analysis                                                                                                                                     |
| 2020 | TableOne                                     | https://esm.uoi.gr/shiny/tableone/                                                         | NA                                                                                                                | 10.1136/ebmental-2020-300162      | Evidence based mental health                                      | TableOne: an online web application and R package for summarising and visualising data                                                                                                                   |
| 2020 | TarPan                                       | https://tarpan.shinyapps.io/tarpan/                                                        | https://github.com/tcashby/tarpan                                                                                 | 10.1186/s12859-020-3477-y         | BMC Bioinformatics                                                | TarPan: an easily adaptable targeted sequencing panel viewer for research and clinical use                                                                                                               |
| 2020 | TippingSens                                  | https://tippingsens.shinyapps.io/TSApp                                                     | https://github.com/CaroHaensch/TippingSensApp                                                                     | NA                                | IDEAS                                                             | TippingSens: An R Shiny Application to Facilitate Sensitivity Analysis for Causal Inference Under Confounding                                                                                            |
| 2020 | TRIP                                         | NA                                                                                         | https://github.com/BiodataAnalysisGroup/TRIP                                                                      | 10.1186/s12859-020-03669-1        | BMC Bioinformatics                                                | TRIP - T cell receptor/immunoglobulin profiler                                                                                                                                                           |
| 2020 | vcfView                                      | NA                                                                                         | https://github.com/BrianOSullivanGit/vcfView                                                                      | 10.1177/1176935120972377          | Cancer Informatics                                                | vcfView: An Extensible Data Visualization and Quality Assurance Platform for Integrated Somatic Variant Analysis                                                                                         |
| 2020 | VDJView                                      | https://bitbucket.org/kirbyvisp/vdjview                                                    | NA                                                                                                                | 10.1186/s12920-020-0696-z         | BMC medical genomics                                              | Exploring and analysing single cell multi-omics data with VDJView                                                                                                                                        |
| 2020 | viscover                                     | https://lyux.shinyapps.io/viscover                                                         | https://github.com/XiaodanLyu/viscover                                                                            | 10.1002/bimj.202000029            | Biometrical journal                                               | Empirical Bayes small area prediction under a zero-inflated lognormal model with correlated random area effects                                                                                          |
| 2020 | VolcaNoseR                                   | https://huygens.science.uva.nl/VolcaNoseR                                                  | https://github.com/JoachimGoedhart/VolcaNoseR                                                                     | 10.1038/s41598-020-76603-3        | Scientific Reports                                                | VolcaNoseR is a web app for creating, exploring, labeling and sharing volcano plots                                                                                                                      |
| 2020 | V-SVA                                        | https://vsva.jax.org/                                                                      | https://github.com/nlawlor/V-SVA                                                                                  | 10.1093/bioinformatics/btaa128    | Bioinformatics                                                    | V-SVA: an R Shiny application for detecting and annotating hidden sources of variation in single-cell RNA-seq data                                                                                       |
| 2020 | webGQT                                       | https://vm1138.kaj.pouta.csc.fi/webgqt/                                                    | https://github.com/arumds/webgqt                                                                                  | 10.3389/fgene.2020.00152          | Frontiers in Genetics                                             | webGQT: A Shiny Server for Genotype Query Tools for Model-Based Variant Filtering                                                                                                                        |
| 2020 | WHONDRS-GUI                                  | https://xmlin.shinyapps.io/whondrs/                                                        | NA                                                                                                                | 10.7717/peerj.9277                | PeerJ                                                             | WHONDRS-GUI: a web application for global survey of surface water metabolites                                                                                                                            |
| 2020 | ZenoFishDb                                   | https://konulab.shinyapps.io/zenofishdb/                                                   | NA                                                                                                                | 10.1089/zeb.2020.1869             | Zebra fish                                                        | ZenoFishDb v1.1: A Database for Xenotransplantation Studies in Zebrafish                                                                                                                                 |
| 2020 | Dark Kinase Knowledgebase                    | https://darkkinome.org/                                                                    | https://github.com/IDG-Kinase/darkkinasekb                                                                        | 10.1093/nar/gkaa853               | Nucleic Acids Research                                            | The Dark Kinase Knowledgebase: an online compendium of knowledge and experimental results of understudied kinases                                                                                        |
| 2021 | AIV_Classifier                               | https://boyuan.shinyapps.io/AIV_Classifier/                                                | NA                                                                                                                | 10.1016/j.chroma.2020.461733      | Journal of chromatography                                         | Free amino acids in African indigenous vegetables: Analysis with improved hydrophilic interaction ultra-high performance liquid chromatography tandem mass spectrometry and interactive machine learning |
| 2021 | AMKdose                                      | NA                                                                                         | http://shiny.cumulo.usal.es/amkdose/                                                                              | 10.1093/jac/dkaa158               | The Journal of antimicrobial chemotherapy                         | Amikacin initial dosage in patients with hypoalbuminaemia: an interactive tool based on a population pharmacokinetic approach                                                                            |
| 2021 | anem                                         | http://gopalpenny.shinyapps.io/anemapp                                                     | http://github.com/gopalpenny/anem                                                                                 | 10.1111/gwat.13043                | Ground water                                                      | anem: A Simple Web-Based Platform to Build Stakeholder Understanding of Groundwater Behavior                                                                                                             |
| 2021 | animalcules                                  | NA                                                                                         | https://github.com/compbiomed/animalcules                                                                         | 10.1186/s40168-021-01013-0        | Microbiome                                                        | animalcules: interactive microbiome analytics and visualization in R                                                                                                                                     |
| 2021 | Arena3Dweb                                   | http://bib.fleming.gr:3838/Arena3D                                                         | https://github.com/PavlopoulosLab/Arena3Dweb                                                                      | 10.1093/nar/gkab278               | Nucleic Acids Research                                            | Arena3Dweb: interactive 3D visualization of multilayered networks                                                                                                                                        |
| 2021 | AVI-Score                                    | https://r-ubesp.dctv.unipd.it/shiny/AVI-Score/                                             | NA                                                                                                                | 10.1002/path.5653                 | The Journal of pathology                                          | Machine learning-based analysis of alveolar and vascular injury in SARS-CoV-2 acute respiratory failure                                                                                                  |
| 2021 | BatchServer                                  | https://lifeinfor.shinyapps.io/batchserver/                                                | https://github.com/guomics-lab/batch_server                                                                       | 10.1021/acs.jproteome.0c00488     | Journal Proteome Research                                         | BatchServer: A Web Server for Batch Effect Evaluation, Visualization, and Correction                                                                                                                     |
| 2021 | CATALYST                                     | NA                                                                                         | https://github.com/HelenaLC/CATALYST                                                                              | 10.1007/s11899-020-00602-4        | Current hematologic malignancy reports                            | High-Dimensional Immune Monitoring for Chimeric Antigen Receptor T Cell Therapies                                                                                                                        |
| 2021 | cell type mixtures                           | NA                                                                                         | https://github.com/yunzhang813/simDeNet-R-Package-Shiny                                                           | 10.1093/bib/bbz135                | Briefings in bioinformatics                                       | The effect of tissue composition on gene co-expression                                                                                                                                                   |
| 2021 | CeNet Omnibus                                | NA                                                                                         | https://github.com/GaoLabXDU/CeNetOmnibus                                                                         | 10.1186/s12859-021-04012-y        | BMC Bioinformatics                                                | CeNet Omnibus: an R/Shiny application to the construction and analysis of competing endogenous RNA network                                                                                               |
| 2021 | Chainchecker                                 | https://shiny.dide.imperial.ac.uk/chainchecker                                             | https://github.com/imperialebola2018/chainchecker                                                                 | 10.1371/journal.pone.0247002      | Plos One                                                          | Chainchecker: An application to visualise and explore transmission chains for Ebola virus disease                                                                                                        |
| 2021 | Correlation AnalyzeR                         | https://gccri.bishop-lab.uthscsa.edu/correlation-analyzer/                                 | https://github.com/Bishop-Laboratory/correlationAnalyzeR                                                          | 10.1186/s12859-021-04130-7        | BMC Bioinformatics                                                | Correlation AnalyzeR: functional predictions from gene co-expression correlations                                                                                                                        |
| 2021 | CrossValTutorial                             | https://qchelseasong.shinyapps.io/CrossValTutorial/                                        | https://osf.io/m62sh/                                                                                             | 10.1177/2515245920947067          | Advances in Methods and Practices in Psychological Science        | Making Sense of Model Generalizability: A Tutorial on Cross-Validation in R and Shiny                                                                                                                    |
| 2021 | DoseResponseDesigns                          | http://biostatistics.dkfz.de/DoseResponseDesigns/                                          | NA                                                                                                                | 10.1016/j.toxlet.2020.11.018      | Toxicology Letters                                                | An R-shiny application to calculate optimal designs for single substance and interaction trials in dose response experiments                                                                             |
| 2021 | Eatomics                                     | https://we.analyzegenomes.com/now/eatomics/                                                | https://github.com/Millchmaedchen/Eatomics                                                                        | 10.1021/acs.jproteome.0c00398     | Journal of Proteome Research                                      | Eatomics: Shiny Exploration of Quantitative Proteomics Data                                                                                                                                              |
| 2021 | EigenGWAS                                    | www.eigengwas.com                                                                          | NA                                                                                                                | 10.1111/1755-0998.13370           | Molecular Ecology Resources                                       | EigenGWAS: An online visualizing and interactive application for detecting genomic signatures of natural selection                                                                                       |
| 2021 | Food Safety Sampling                         | https://go.illinois.edu/FoodSafetySampling                                                 | https://github.com/ericxbcheng/ILSI_remote_repo                                                                   | 10.1111/risa.13721                | Risk analysis                                                     | Evaluation of the Impact of Skewness, Clustering, and Probe Sampling Plan on Aflatoxin Detection in Corn                                                                                                 |
| 2021 | Gene-PROBER                                  | http://gene-prober.icbm.de/                                                                | NA                                                                                                                | 10.1016/j.syapm.2020.126173       | Systematic and applied microbiology                               | Gene-PROBER - a tool to design polynucleotide probes for targeting microbial genes                                                                                                                       |
| 2021 | HCMMCNVs                                     | NA                                                                                         | https://github.com/lunching/HCMM_CNVs                                                                             | 10.1093/bioinformatics/btab183    | Bioinformatics                                                    | HCMMCNVs: Hierarchical Clustering Mixture Model of Copy Number Variants Detection using Whole Exome Sequencing Technology                                                                                |
| 2021 | IPDmada                                      | https://jwang7.shinyapps.io/ipdmada/                                                       | NA                                                                                                                | 10.1002/jrsm.1444                 | Research synthesis methods                                        | IPDmada: An R Shiny tool for analyzing and visualizing individual patient data meta-analyses of diagnostic test accuracy                                                                                 |
| 2021 | ISOFAST                                      | https://analytics.iasoybeans.com/cool-apps/ISOFAST/                                        | NA                                                                                                                | 10.1002/jrsm.1440                 | Research Synthesis Methods                                        | Interactive Web-based Data Visualization and Analysis Tool for Synthetizing on-farm Research Networks Data                                                                                               |
| 2021 | MegaR                                        | https://megar.shinyapps.io/preview/                                                        | https://github.com/BioHPC/MegaR                                                                                   | 10.1186/s12859-020-03933-4        | BMC Bioinformatics                                                | MegaR: an interactive R package for rapid sample classification and phenotype prediction using metagenome profiles and machine learning                                                                  |
| 2021 | MiBiOmics                                    | https://shiny-bird.univ-nantes.fr/app/Mibiomics                                            | https://gitlab.univ-nantes.fr/combi-ls2n/mibiomics                                                                | 10.1186/s12859-020-03921-8        | BMC Bioinformatics                                                | MiBiOmics: an interactive web application for multi‑omics data exploration and integration                                                                                                               |
| 2021 | missing_data_CRT                             | https://utrecht-university.shinyapps.io/missing_data_CRT/                                  | NA                                                                                                                | 10.3758/s13428-020-01529-7        | Behavior Research Methods                                         | The effect of missing data on design efficiency in repeated cross-sectional multi-period two-arm parallel cluster randomized trials                                                                      |
| 2021 | MuSA                                         | NA                                                                                         | https://gitlab.com/Zanfardino/musa                                                                                | 10.1038/s41598-021-81200-z        | Scientific reports                                                | MuSA: a graphical user interface for multi-OMICs data integration in radiogenomic studies                                                                                                                |
| 2021 | network meta‐analyses                        | https://cinema.ispm.unibe.ch/shinies/GRISELDA/                                             | https://github.com/y‐luo06/cNMA_of_antidepressant                                                                 | 10.1002/jrsm.1413                 | Research Synthesis Methods                                        | Visualizing the evolution of evidence: Cumulative network meta-analyses of new generation antidepressants in the last 40 years                                                                           |
| 2021 | PCAT                                         | http://pcat.zhenglab.info                                                                  | NA                                                                                                                | 10.1093/nar/gkaa698               | Nucleic Acids Research                                            | PCAT: an integrated portal for genomic and preclinical testing data of pediatric cancer patient-derived xenograft models                                                                                 |
| 2021 | PhenoSpace                                   | https://shiny.cefe.cnrs.fr/PhenoSpace/                                                     | https://github.com/jsegrestin/PhenoSpace                                                                          | 10.1002/ece3.6928                 | Ecology and Evolution                                             | PhenoSpace: A Shiny application to visualize trait data in the phenotypic space of the global spectrum of plant form and function                                                                        |
| 2021 | ProgPerm                                     | https://biostatistics.mdanderson.org/shinyapps/ProgPerm                                    | https://github.com/LyonsZhang/ProgPerm                                                                            | 10.1186/s12859-021-04061-3        | BMC Bioinformatics                                                | ProgPerm: Progressive permutation for a dynamic representation of the robustness of microbiome discoveries                                                                                               |
| 2021 | PSpecteR                                     | https://hub.docker.com/r/emslcomputing/pspecter                                            | https://github.com/EMSL-Computing/PSpecteR                                                                        | 10.1021/acs.jproteome.0c00857     | Journal of Proteome Research                                      | PSpecteR: A User-Friendly and Interactive Application for Visualizing Top-Down and Bottom-Up Proteomics Data in R                                                                                        |
| 2021 | sangeranalyseR                               | NA                                                                                         | https://github.com/roblanf/sangeranalyseR                                                                         | 10.1093/gbe/evab028               | Genome Biology and Evolution                                      | sangeranalyseR: Simple and Interactive Processing of Sanger Sequencing Data in R                                                                                                                         |
| 2021 | SAPHIR                                       | NA                                                                                         | https://github.com/elodiegermani/SAPHIR                                                                           | 10.12688/f1000research.27062.2    | F1000Research                                                     | SAPHIR: a Shiny application to analyze tissue section images                                                                                                                                             |
| 2021 | Shiny Crypto Assets                          | http://data.bitcoinity.org/                                                                | NA                                                                                                                | 10.2139/ssrn.3796837              | SSRN                                                              | Shiny' Crypto Assets: A Systemic Look at Gold-Backed Cryptocurrencies during the COVID-19 Pandemic                                                                                                       |
| 2021 | STRait Razor Online                          | https://www.unthsc.edu/straitrazor                                                         | https://github.com/ExpectationsManaged/STRaitRazorOnline.                                                         | 10.1016/j.fsigen.2021.102463      | Forensic science international. Genetics                          | STRait Razor Online: An enhanced user interface to facilitate interpretation of MPS data                                                                                                                 |
| 2021 | TREND-DB                                     | http://shiny.imbei.unimainz.de:3838/trend-db                                               | https://github.com/federicomarini/trend-db                                                                        | 10.1093/nar/gkaa722               | Nucleic Acids Research                                            | TREND-DB––a transcriptome-wide atlas of the dynamic landscape of alternative polyadenylation                                                                                                             |
| 2021 | viromeBrowser                                | NA                                                                                         | https://cran.r-project.org/package=viromeBrowser                                                                  | 10.3390/v13030437                 | Viruses                                                           | viromeBrowser: A Shiny App for Browsing Virome Sequencing Analysis Results                                                                                                                               |
| 2021 | WASP                                         | http://www.computational.bio/software/wasp                                                 | https://github.com/andreashoek/wasp                                                                               | 10.1186/s12864-021-07469-6        | BMC Genomics                                                      | WASP: a versatile, web-accessible single cell RNA-Seq processing platform                                                                                                                                |
| 2021 | wiSDOM                                       | https://lun-ching.shinyapps.io/wisdom/                                                     | https://github.com/lunching/wiSDOM                                                                                | 10.1093/bioinformatics/btab057    | Bioinformatics                                                    | wiSDOM: a visual and statistical analytics for interrogating microbiome                                                                                                                                  |
| 2021 | Arena3Dweb                                   | http://bib.fleming.gr:3838/Arena3D                                                         | https://github.com/PavlopoulosLab/Arena3Dweb                                                                      | 10.1093/nar/gkab278               | Nucleic Acids Research                                            | Arena3Dweb: interactive 3D visualization of multilayered networks                                                                                                                                        |
| 2021 | GeneTonic                                    | http://shiny.imbei.uni-mainz.de:3838/GeneTonic                                             | https://github.com/federicomarini/GeneTonic                                                                       | 10.1101/2021.05.19.444862         | bioRxiv                                                           | GeneTonic: an R/Bioconductor package for streamlining the interpretation of RNA-seq data                                                                                                                 |
| 2021 | ProteomeExpert                               | https://proteomic.shinyapps.io/peserver/                                                   | http://www.github.com/lifeinfo/ProteomeExpert/                                                                    | 10.1093/bioinformatics/btaa1088   | Bioinformatics                                                    | ProteomeExpert: a Docker image-based web server for exploring, modeling, visualizing and mining quantitative proteomic datasets                                                                          |
| 2021 | powdR                                        | https://cran.r-project.org/package=powdR                                                   | https://github.com/benmbutler/powdR                                                                               | 10.1016/j.cageo.2020.104662       | Computers & Geosciences                                           | powdR: An R package for quantitative mineralogy using full pattern summation of X-ray powder diffraction data                                                                                            |
| 2021 | spatialLIBD                                  | http://bioconductor.org/packages/release/data/experiment/html/spatialLIBD.html             |                                                                                                                   | 10.1101/2021.04.29.440149         | bioRxiv                                                           | spatialLIBD: an R/Bioconductor package to visualize spatially-resolved transcriptomics data                                                                                                              |
| 2021 | FielDHub                                     | https://didiermurillof.github.io/FielDHub/reference/index.html                             | https://github.com/DidierMurilloF/FielDHub/                                                                       | 10.21105/joss.03122               | Journal of Open Source Software                                   | FielDHub: A Shiny App for Design of Experiments in Life Sciences                                                                                                                                         |
| 2021 | ConTESSA                                     | https://iddynamicsjhu.shinyapps.io/contessa/                                               |                                                                                                                   | 10.1371/journal.pmed.1003585      | PLOS Medicine                                                     | Maximizing and evaluating the impact of test-trace-isolate programs: A modeling study                                                                                                                    |
